#0 building with "default" instance using docker driver

#1 [internal] load .dockerignore
#1 ...

#2 [internal] load build definition from v1_Dockerfile
#2 transferring dockerfile: 235B done
#2 DONE 0.4s

#1 [internal] load .dockerignore
#1 transferring context: 54B 0.0s done
#1 DONE 0.5s

#3 [internal] load metadata for docker.io/library/node:16
#3 DONE 1.1s

#4 [1/6] FROM docker.io/library/node:16@sha256:f77a1aef2da8d83e45ec990f45df50f1a286c5fe8bbfb8c6e4246c6389705c0b
#4 resolve docker.io/library/node:16@sha256:f77a1aef2da8d83e45ec990f45df50f1a286c5fe8bbfb8c6e4246c6389705c0b
#4 resolve docker.io/library/node:16@sha256:f77a1aef2da8d83e45ec990f45df50f1a286c5fe8bbfb8c6e4246c6389705c0b 0.2s done
#4 sha256:f77a1aef2da8d83e45ec990f45df50f1a286c5fe8bbfb8c6e4246c6389705c0b 776B / 776B done
#4 sha256:c94b82f9827cab6e421b350965a9ef11b25b13ffbd1030536203d541f55dcbe2 2.00kB / 2.00kB done
#4 sha256:1ddc7e4055fdb6f6bf31063b593befda814294f9f904b6ddfc21ab1513bafa8e 7.23kB / 7.23kB done
#4 sha256:311da6c465ea1576925360eba391bcd32dece9be95960a0bc9ffcb25fe712017 0B / 50.50MB 0.1s
#4 ...

#5 [internal] load build context
#5 transferring context: 3.35MB 0.2s done
#5 DONE 0.5s

#4 [1/6] FROM docker.io/library/node:16@sha256:f77a1aef2da8d83e45ec990f45df50f1a286c5fe8bbfb8c6e4246c6389705c0b
#4 sha256:7e9bf114588c05b2df612b083b96582f3b8dbf51647aa6138a50d09d42df2454 0B / 17.58MB 0.2s
#4 sha256:311da6c465ea1576925360eba391bcd32dece9be95960a0bc9ffcb25fe712017 13.63MB / 50.50MB 0.5s
#4 sha256:7e9bf114588c05b2df612b083b96582f3b8dbf51647aa6138a50d09d42df2454 4.19MB / 17.58MB 0.5s
#4 sha256:ffd9397e94b74abcb54e514f1430e00f604328d1f895eadbd482f08cc02444e5 0B / 51.89MB 0.5s
#4 sha256:311da6c465ea1576925360eba391bcd32dece9be95960a0bc9ffcb25fe712017 19.92MB / 50.50MB 0.6s
#4 sha256:7e9bf114588c05b2df612b083b96582f3b8dbf51647aa6138a50d09d42df2454 9.44MB / 17.58MB 0.6s
#4 sha256:311da6c465ea1576925360eba391bcd32dece9be95960a0bc9ffcb25fe712017 25.17MB / 50.50MB 0.7s
#4 sha256:7e9bf114588c05b2df612b083b96582f3b8dbf51647aa6138a50d09d42df2454 13.63MB / 17.58MB 0.7s
#4 sha256:ffd9397e94b74abcb54e514f1430e00f604328d1f895eadbd482f08cc02444e5 4.19MB / 51.89MB 0.7s
#4 sha256:311da6c465ea1576925360eba391bcd32dece9be95960a0bc9ffcb25fe712017 30.41MB / 50.50MB 0.9s
#4 sha256:7e9bf114588c05b2df612b083b96582f3b8dbf51647aa6138a50d09d42df2454 17.58MB / 17.58MB 0.9s
#4 sha256:ffd9397e94b74abcb54e514f1430e00f604328d1f895eadbd482f08cc02444e5 14.68MB / 51.89MB 0.9s
#4 sha256:311da6c465ea1576925360eba391bcd32dece9be95960a0bc9ffcb25fe712017 34.60MB / 50.50MB 1.0s
#4 sha256:ffd9397e94b74abcb54e514f1430e00f604328d1f895eadbd482f08cc02444e5 19.92MB / 51.89MB 1.0s
#4 sha256:311da6c465ea1576925360eba391bcd32dece9be95960a0bc9ffcb25fe712017 40.89MB / 50.50MB 1.1s
#4 sha256:7e9bf114588c05b2df612b083b96582f3b8dbf51647aa6138a50d09d42df2454 17.58MB / 17.58MB 1.0s done
#4 sha256:ffd9397e94b74abcb54e514f1430e00f604328d1f895eadbd482f08cc02444e5 25.17MB / 51.89MB 1.1s
#4 sha256:311da6c465ea1576925360eba391bcd32dece9be95960a0bc9ffcb25fe712017 50.33MB / 50.50MB 1.3s
#4 sha256:ffd9397e94b74abcb54e514f1430e00f604328d1f895eadbd482f08cc02444e5 35.65MB / 51.89MB 1.3s
#4 sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 0B / 191.90MB 1.3s
#4 sha256:ffd9397e94b74abcb54e514f1430e00f604328d1f895eadbd482f08cc02444e5 39.85MB / 51.89MB 1.4s
#4 sha256:ffd9397e94b74abcb54e514f1430e00f604328d1f895eadbd482f08cc02444e5 45.09MB / 51.89MB 1.4s
#4 sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 11.53MB / 191.90MB 1.4s
#4 sha256:ffd9397e94b74abcb54e514f1430e00f604328d1f895eadbd482f08cc02444e5 51.89MB / 51.89MB 1.7s
#4 sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 24.12MB / 191.90MB 1.7s
#4 sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 40.89MB / 191.90MB 1.9s
#4 sha256:311da6c465ea1576925360eba391bcd32dece9be95960a0bc9ffcb25fe712017 50.50MB / 50.50MB 2.0s done
#4 sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 52.43MB / 191.90MB 2.2s
#4 sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 63.96MB / 191.90MB 2.4s
#4 extracting sha256:311da6c465ea1576925360eba391bcd32dece9be95960a0bc9ffcb25fe712017
#4 sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 75.50MB / 191.90MB 2.6s
#4 sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 87.03MB / 191.90MB 2.8s
#4 sha256:ffd9397e94b74abcb54e514f1430e00f604328d1f895eadbd482f08cc02444e5 51.89MB / 51.89MB 2.8s done
#4 sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 97.52MB / 191.90MB 2.9s
#4 sha256:0e421f66aff42bb069dffc26af6d132194b22a1082b08c5ef7cd69c627783c04 0B / 34.79MB 2.9s
#4 sha256:ae3b95bbaa61ce24cefdd89e7c74d6fbd7713b2bcae93af47063d06bd7e02172 0B / 4.20kB 3.1s
#4 sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 108.00MB / 191.90MB 3.1s
#4 sha256:0e421f66aff42bb069dffc26af6d132194b22a1082b08c5ef7cd69c627783c04 6.29MB / 34.79MB 3.1s
#4 sha256:ae3b95bbaa61ce24cefdd89e7c74d6fbd7713b2bcae93af47063d06bd7e02172 4.20kB / 4.20kB 3.1s done
#4 sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 119.54MB / 191.90MB 3.3s
#4 sha256:0e421f66aff42bb069dffc26af6d132194b22a1082b08c5ef7cd69c627783c04 17.83MB / 34.79MB 3.3s
#4 sha256:ca266fd6192108b67fb57b74753a8c4ca5d8bd458baae3d4df7ce9f42dedcc1d 0B / 2.27MB 3.3s
#4 sha256:0e421f66aff42bb069dffc26af6d132194b22a1082b08c5ef7cd69c627783c04 23.07MB / 34.79MB 3.4s
#4 sha256:ca266fd6192108b67fb57b74753a8c4ca5d8bd458baae3d4df7ce9f42dedcc1d 2.27MB / 2.27MB 3.4s
#4 sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 134.22MB / 191.90MB 3.7s
#4 sha256:0e421f66aff42bb069dffc26af6d132194b22a1082b08c5ef7cd69c627783c04 34.79MB / 34.79MB 3.7s
#4 sha256:ca266fd6192108b67fb57b74753a8c4ca5d8bd458baae3d4df7ce9f42dedcc1d 2.27MB / 2.27MB 3.5s done
#4 sha256:ee7d78be1eb92caf6ae84fc3af736b23eca018d5dedc967ae5bdee6d7082403b 0B / 450B 3.7s
#4 sha256:ee7d78be1eb92caf6ae84fc3af736b23eca018d5dedc967ae5bdee6d7082403b 450B / 450B 3.8s
#4 sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 145.75MB / 191.90MB 3.9s
#4 sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 162.53MB / 191.90MB 4.2s
#4 sha256:0e421f66aff42bb069dffc26af6d132194b22a1082b08c5ef7cd69c627783c04 34.79MB / 34.79MB 4.2s done
#4 sha256:ee7d78be1eb92caf6ae84fc3af736b23eca018d5dedc967ae5bdee6d7082403b 450B / 450B 4.2s done
#4 sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 175.11MB / 191.90MB 4.3s
#4 sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 187.70MB / 191.90MB 4.5s
#4 extracting sha256:311da6c465ea1576925360eba391bcd32dece9be95960a0bc9ffcb25fe712017 5.2s
#4 sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 191.90MB / 191.90MB 7.5s done
#4 extracting sha256:311da6c465ea1576925360eba391bcd32dece9be95960a0bc9ffcb25fe712017 9.7s done
#4 extracting sha256:7e9bf114588c05b2df612b083b96582f3b8dbf51647aa6138a50d09d42df2454 0.1s
#4 extracting sha256:7e9bf114588c05b2df612b083b96582f3b8dbf51647aa6138a50d09d42df2454 1.8s done
#4 extracting sha256:ffd9397e94b74abcb54e514f1430e00f604328d1f895eadbd482f08cc02444e5
#4 extracting sha256:ffd9397e94b74abcb54e514f1430e00f604328d1f895eadbd482f08cc02444e5 5.0s
#4 extracting sha256:ffd9397e94b74abcb54e514f1430e00f604328d1f895eadbd482f08cc02444e5 8.1s done
#4 extracting sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e
#4 extracting sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 5.2s
#4 extracting sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 10.2s
#4 extracting sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 15.3s
#4 extracting sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 20.4s
#4 extracting sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 27.8s
#4 extracting sha256:513d779256048c961239af5f500589330546b072775217272e19ffae1635e98e 32.1s done
#4 extracting sha256:ae3b95bbaa61ce24cefdd89e7c74d6fbd7713b2bcae93af47063d06bd7e02172
#4 extracting sha256:ae3b95bbaa61ce24cefdd89e7c74d6fbd7713b2bcae93af47063d06bd7e02172 0.4s done
#4 extracting sha256:0e421f66aff42bb069dffc26af6d132194b22a1082b08c5ef7cd69c627783c04
#4 extracting sha256:0e421f66aff42bb069dffc26af6d132194b22a1082b08c5ef7cd69c627783c04 5.0s done
#4 extracting sha256:ca266fd6192108b67fb57b74753a8c4ca5d8bd458baae3d4df7ce9f42dedcc1d
#4 extracting sha256:ca266fd6192108b67fb57b74753a8c4ca5d8bd458baae3d4df7ce9f42dedcc1d 0.6s done
#4 extracting sha256:ee7d78be1eb92caf6ae84fc3af736b23eca018d5dedc967ae5bdee6d7082403b 0.0s done
#4 DONE 65.6s

#6 [2/6] RUN mkdir -p /usr/src/app
#6 DONE 14.8s

#7 [3/6] WORKDIR /usr/src/app
#7 DONE 1.4s

#8 [4/6] COPY package.json /usr/src/app
#8 DONE 0.4s

#9 [5/6] RUN npm install && npm cache clean --force
#9 65.71 npm WARN deprecated formidable@1.2.6: Please upgrade to latest, formidable@v2 or formidable@v3! Check these notes: https://bit.ly/2ZEqIau
#9 65.71 npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
#9 66.08 npm WARN deprecated cryptiles@2.0.5: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
#9 66.25 npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
#9 66.61 npm WARN deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
#9 66.62 npm WARN deprecated formatio@1.2.0: This package is unmaintained. Use @sinonjs/formatio instead
#9 68.49 npm WARN deprecated boom@2.10.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
#9 68.49 npm WARN deprecated samsam@1.3.0: This package has been deprecated in favour of @sinonjs/samsam
#9 68.52 npm WARN deprecated caller-id@0.1.0: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
#9 68.52 npm WARN deprecated json3@3.3.2: Please use the native JSON object instead of JSON 3
#9 69.09 npm WARN deprecated sntp@1.0.9: This module moved to @hapi/sntp. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.
#9 70.49 npm WARN deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
#9 73.40 npm WARN deprecated har-validator@2.0.6: this library is no longer supported
#9 76.38 npm WARN deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
#9 76.43 npm WARN deprecated rimraf@2.7.1: Rimraf versions prior to v4 are no longer supported
#9 76.43 npm WARN deprecated circular-json@0.3.3: CircularJSON is in maintenance only, flatted is its successor.
#9 78.59 npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
#9 80.46 npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
#9 80.75 npm WARN deprecated hoek@2.16.3: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
#9 80.81 npm WARN deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
#9 81.34 npm WARN deprecated koa-router@7.4.0: **IMPORTANT 10x+ PERFORMANCE UPGRADE**: Please upgrade to v12.0.1+ as we have fixed an issue with debuglog causing 10x slower router benchmark performance, see https://github.com/koajs/router/pull/173
#9 82.63 npm WARN deprecated rimraf@2.6.3: Rimraf versions prior to v4 are no longer supported
#9 82.71 npm WARN deprecated argv@0.0.2: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
#9 83.21 npm WARN deprecated glob@7.0.5: Glob versions prior to v9 are no longer supported
#9 84.17 npm WARN deprecated request@2.77.0: request has been deprecated, see https://github.com/request/request/issues/3142
#9 84.32 npm WARN deprecated mkdirp@0.5.1: Legacy versions of mkdirp are no longer supported. Please update to mkdirp 1.x. (Note that the API surface has changed to use Promises in 1.x.)
#9 84.55 npm WARN deprecated text-encoding@0.6.4: no longer maintained
#9 85.00 npm WARN deprecated hawk@3.1.3: This module moved to @hapi/hawk. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.
#9 85.15 npm WARN deprecated superagent@3.8.3: Please upgrade to v9.0.0+ as we have fixed a public vulnerability with formidable dependency. Note that v9.0.0+ requires Node.js v14.18.0+. See https://github.com/ladjs/superagent/pull/1800 for insight. This project is supported and maintained by the team at Forward Email @ https://forwardemail.net
#9 93.99 npm WARN deprecated sinon@2.4.1: 16.1.1
#9 101.7 npm WARN deprecated core-js@2.6.12: core-js@<3.23.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Some versions have web compatibility issues. Please, upgrade your dependencies to the actual version of core-js.
#9 425.6 npm notice 
#9 425.6 npm notice New major version of npm available! 8.19.4 -> 10.8.2
#9 425.6 npm notice Changelog: <https://github.com/npm/cli/releases/tag/v10.8.2>
#9 425.6 npm notice Run `npm install -g npm@10.8.2` to update!
#9 425.6 npm notice 
#9 425.6 npm ERR! code 1
#9 425.6 npm ERR! path /usr/src/app/node_modules/sqlite3
#9 425.6 npm ERR! command failed
#9 425.6 npm ERR! command sh -c -- node-pre-gyp install --fallback-to-build
#9 425.6 npm ERR! make: Entering directory '/usr/src/app/node_modules/sqlite3/build'
#9 425.6 npm ERR!   ACTION deps_sqlite3_gyp_action_before_build_target_unpack_sqlite_dep Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c
#9 425.6 npm ERR! make: Entering directory '/usr/src/app/node_modules/sqlite3/build'
#9 425.6 npm ERR!   ACTION deps_sqlite3_gyp_action_before_build_target_unpack_sqlite_dep Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c
#9 425.6 npm ERR!   TOUCH Release/obj.target/deps/action_before_build.stamp
#9 425.6 npm ERR!   CC(target) Release/obj.target/sqlite3/gen/sqlite-autoconf-3150000/sqlite3.o
#9 425.6 npm ERR!   TOUCH Release/obj.target/deps/action_before_build.stamp
#9 425.6 npm ERR!   CC(target) Release/obj.target/sqlite3/gen/sqlite-autoconf-3150000/sqlite3.o
#9 425.6 npm ERR!   AR(target) Release/obj.target/deps/sqlite3.a
#9 425.6 npm ERR!   COPY Release/sqlite3.a
#9 425.6 npm ERR!   CXX(target) Release/obj.target/node_sqlite3/src/database.o
#9 425.6 npm ERR! make: Leaving directory '/usr/src/app/node_modules/sqlite3/build'
#9 425.6 npm ERR! Failed to execute '/usr/local/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/usr/src/app/node_modules/sqlite3/lib/binding/node-v93-linux-x64/node_sqlite3.node --module_name=node_sqlite3 --module_path=/usr/src/app/node_modules/sqlite3/lib/binding/node-v93-linux-x64' (1)
#9 425.6 npm ERR!   AR(target) Release/obj.target/deps/sqlite3.a
#9 425.6 npm ERR!   COPY Release/sqlite3.a
#9 425.6 npm ERR!   CXX(target) Release/obj.target/node_sqlite3/src/database.o
#9 425.6 npm ERR! make: Leaving directory '/usr/src/app/node_modules/sqlite3/build'
#9 425.6 npm ERR! node-pre-gyp info it worked if it ends with ok
#9 425.6 npm ERR! node-pre-gyp info using node-pre-gyp@0.6.38
#9 425.6 npm ERR! node-pre-gyp info using node@16.20.2 | linux | x64
#9 425.6 npm ERR! node-pre-gyp info check checked for "/usr/src/app/node_modules/sqlite3/lib/binding/node-v93-linux-x64/node_sqlite3.node" (not found)
#9 425.6 npm ERR! node-pre-gyp http GET https://mapbox-node-binary.s3.amazonaws.com/sqlite3/v3.1.13/node-v93-linux-x64.tar.gz
#9 425.6 npm ERR! node-pre-gyp http 403 https://mapbox-node-binary.s3.amazonaws.com/sqlite3/v3.1.13/node-v93-linux-x64.tar.gz
#9 425.6 npm ERR! node-pre-gyp ERR! Tried to download(403): https://mapbox-node-binary.s3.amazonaws.com/sqlite3/v3.1.13/node-v93-linux-x64.tar.gz 
#9 425.6 npm ERR! node-pre-gyp ERR! Pre-built binaries not found for sqlite3@3.1.13 and node@16.20.2 (node-v93 ABI) (falling back to source compile with node-gyp) 
#9 425.6 npm ERR! node-pre-gyp http 403 status code downloading tarball https://mapbox-node-binary.s3.amazonaws.com/sqlite3/v3.1.13/node-v93-linux-x64.tar.gz 
#9 425.6 npm ERR! node-pre-gyp ERR! Tried to download(undefined): https://mapbox-node-binary.s3.amazonaws.com/sqlite3/v3.1.13/node-v93-linux-x64.tar.gz 
#9 425.6 npm ERR! node-pre-gyp ERR! Pre-built binaries not found for sqlite3@3.1.13 and node@16.20.2 (node-v93 ABI) (falling back to source compile with node-gyp) 
#9 425.6 npm ERR! node-pre-gyp http Connection closed while downloading tarball file 
#9 425.6 npm ERR! gyp info it worked if it ends with ok
#9 425.6 npm ERR! gyp info using node-gyp@9.1.0
#9 425.6 npm ERR! gyp info using node@16.20.2 | linux | x64
#9 425.6 npm ERR! gyp info ok 
#9 425.6 npm ERR! gyp info it worked if it ends with ok
#9 425.6 npm ERR! gyp info using node-gyp@9.1.0
#9 425.6 npm ERR! gyp info using node@16.20.2 | linux | x64
#9 425.6 npm ERR! gyp info ok 
#9 425.6 npm ERR! gyp info it worked if it ends with ok
#9 425.6 npm ERR! gyp info using node-gyp@9.1.0
#9 425.6 npm ERR! gyp info using node@16.20.2 | linux | x64
#9 425.6 npm ERR! gyp info it worked if it ends with ok
#9 425.6 npm ERR! gyp info using node-gyp@9.1.0
#9 425.6 npm ERR! gyp info using node@16.20.2 | linux | x64
#9 425.6 npm ERR! gyp info find Python using Python version 3.7.3 found at "/usr/bin/python3"
#9 425.6 npm ERR! gyp info find Python using Python version 3.7.3 found at "/usr/bin/python3"
#9 425.6 npm ERR! gyp http GET https://nodejs.org/download/release/v16.20.2/node-v16.20.2-headers.tar.gz
#9 425.6 npm ERR! gyp http GET https://nodejs.org/download/release/v16.20.2/node-v16.20.2-headers.tar.gz
#9 425.6 npm ERR! gyp http 200 https://nodejs.org/download/release/v16.20.2/node-v16.20.2-headers.tar.gz
#9 425.6 npm ERR! gyp http 200 https://nodejs.org/download/release/v16.20.2/node-v16.20.2-headers.tar.gz
#9 425.6 npm ERR! gyp http GET https://nodejs.org/download/release/v16.20.2/SHASUMS256.txt
#9 425.6 npm ERR! gyp http 200 https://nodejs.org/download/release/v16.20.2/SHASUMS256.txt
#9 425.6 npm ERR! gyp info spawn /usr/bin/python3
#9 425.6 npm ERR! gyp info spawn args [
#9 425.6 npm ERR! gyp info spawn args   '/usr/local/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
#9 425.6 npm ERR! gyp info spawn args   'binding.gyp',
#9 425.6 npm ERR! gyp info spawn args   '-f',
#9 425.6 npm ERR! gyp info spawn args   'make',
#9 425.6 npm ERR! gyp info spawn args   '-I',
#9 425.6 npm ERR! gyp info spawn args   '/usr/src/app/node_modules/sqlite3/build/config.gypi',
#9 425.6 npm ERR! gyp info spawn args   '-I',
#9 425.6 npm ERR! gyp info spawn args   '/usr/local/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
#9 425.6 npm ERR! gyp info spawn args   '-I',
#9 425.6 npm ERR! gyp info spawn args   '/root/.cache/node-gyp/16.20.2/include/node/common.gypi',
#9 425.6 npm ERR! gyp info spawn args   '-Dlibrary=shared_library',
#9 425.6 npm ERR! gyp info spawn args   '-Dvisibility=default',
#9 425.6 npm ERR! gyp info spawn args   '-Dnode_root_dir=/root/.cache/node-gyp/16.20.2',
#9 425.6 npm ERR! gyp info spawn args   '-Dnode_gyp_dir=/usr/local/lib/node_modules/npm/node_modules/node-gyp',
#9 425.6 npm ERR! gyp info spawn args   '-Dnode_lib_file=/root/.cache/node-gyp/16.20.2/<(target_arch)/node.lib',
#9 425.6 npm ERR! gyp info spawn args   '-Dmodule_root_dir=/usr/src/app/node_modules/sqlite3',
#9 425.6 npm ERR! gyp info spawn args   '-Dnode_engine=v8',
#9 425.6 npm ERR! gyp info spawn args   '--depth=.',
#9 425.6 npm ERR! gyp info spawn args   '--no-parallel',
#9 425.6 npm ERR! gyp info spawn args   '--generator-output',
#9 425.6 npm ERR! gyp info spawn args   'build',
#9 425.6 npm ERR! gyp info spawn args   '-Goutput_dir=.'
#9 425.6 npm ERR! gyp info spawn args ]
#9 425.6 npm ERR! gyp http GET https://nodejs.org/download/release/v16.20.2/SHASUMS256.txt
#9 425.6 npm ERR! gyp http 200 https://nodejs.org/download/release/v16.20.2/SHASUMS256.txt
#9 425.6 npm ERR! gyp info spawn /usr/bin/python3
#9 425.6 npm ERR! gyp info spawn args [
#9 425.6 npm ERR! gyp info spawn args   '/usr/local/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
#9 425.6 npm ERR! gyp info spawn args   'binding.gyp',
#9 425.6 npm ERR! gyp info spawn args   '-f',
#9 425.6 npm ERR! gyp info spawn args   'make',
#9 425.6 npm ERR! gyp info spawn args   '-I',
#9 425.6 npm ERR! gyp info spawn args   '/usr/src/app/node_modules/sqlite3/build/config.gypi',
#9 425.6 npm ERR! gyp info spawn args   '-I',
#9 425.6 npm ERR! gyp info spawn args   '/usr/local/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
#9 425.6 npm ERR! gyp info spawn args   '-I',
#9 425.6 npm ERR! gyp info spawn args   '/root/.cache/node-gyp/16.20.2/include/node/common.gypi',
#9 425.6 npm ERR! gyp info spawn args   '-Dlibrary=shared_library',
#9 425.6 npm ERR! gyp info spawn args   '-Dvisibility=default',
#9 425.6 npm ERR! gyp info spawn args   '-Dnode_root_dir=/root/.cache/node-gyp/16.20.2',
#9 425.6 npm ERR! gyp info spawn args   '-Dnode_gyp_dir=/usr/local/lib/node_modules/npm/node_modules/node-gyp',
#9 425.6 npm ERR! gyp info spawn args   '-Dnode_lib_file=/root/.cache/node-gyp/16.20.2/<(target_arch)/node.lib',
#9 425.6 npm ERR! gyp info spawn args   '-Dmodule_root_dir=/usr/src/app/node_modules/sqlite3',
#9 425.6 npm ERR! gyp info spawn args   '-Dnode_engine=v8',
#9 425.6 npm ERR! gyp info spawn args   '--depth=.',
#9 425.6 npm ERR! gyp info spawn args   '--no-parallel',
#9 425.6 npm ERR! gyp info spawn args   '--generator-output',
#9 425.6 npm ERR! gyp info spawn args   'build',
#9 425.6 npm ERR! gyp info spawn args   '-Goutput_dir=.'
#9 425.6 npm ERR! gyp info spawn args ]
#9 425.6 npm ERR! gyp info ok 
#9 425.6 npm ERR! gyp info ok 
#9 425.6 npm ERR! gyp info it worked if it ends with ok
#9 425.6 npm ERR! gyp info using node-gyp@9.1.0
#9 425.6 npm ERR! gyp info using node@16.20.2 | linux | x64
#9 425.6 npm ERR! gyp info it worked if it ends with ok
#9 425.6 npm ERR! gyp info using node-gyp@9.1.0
#9 425.6 npm ERR! gyp info using node@16.20.2 | linux | x64
#9 425.6 npm ERR! gyp info spawn make
#9 425.6 npm ERR! gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
#9 425.6 npm ERR! gyp info spawn make
#9 425.6 npm ERR! gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
#9 425.6 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'strftimeFunc':
#9 425.6 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.6 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.6 npm ERR!                            ^
#9 425.6 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:19615:52: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.6 npm ERR!                        z==zBuf ? SQLITE_TRANSIENT : SQLITE_DYNAMIC);
#9 425.6 npm ERR!                                                     ^~~~~~~~~~~~~~
#9 425.6 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'strftimeFunc':
#9 425.6 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.6 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.6 npm ERR!                            ^
#9 425.6 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:19615:52: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.6 npm ERR!                        z==zBuf ? SQLITE_TRANSIENT : SQLITE_DYNAMIC);
#9 425.6 npm ERR!                                                     ^~~~~~~~~~~~~~
#9 425.6 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3ErrorWithMsg':
#9 425.6 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.6 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.6 npm ERR!                            ^
#9 425.6 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:27452:54: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.6 npm ERR!      sqlite3ValueSetStr(db->pErr, -1, z, SQLITE_UTF8, SQLITE_DYNAMIC);
#9 425.6 npm ERR!                                                       ^~~~~~~~~~~~~~
#9 425.6 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3ErrorWithMsg':
#9 425.6 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.6 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.6 npm ERR!                            ^
#9 425.6 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:27452:54: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.6 npm ERR!      sqlite3ValueSetStr(db->pErr, -1, z, SQLITE_UTF8, SQLITE_DYNAMIC);
#9 425.6 npm ERR!                                                       ^~~~~~~~~~~~~~
#9 425.6 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3VdbeMemSetStr':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:69652:19: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!    }else if( xDel==SQLITE_DYNAMIC ){
#9 425.7 npm ERR!                    ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'valueFromExpr':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:70058:55: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!        sqlite3ValueSetStr(pVal, -1, zVal, SQLITE_UTF8, SQLITE_DYNAMIC);
#9 425.7 npm ERR!                                                        ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:70100:29: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!                           0, SQLITE_DYNAMIC);
#9 425.7 npm ERR!                              ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3VdbeMemSetStr':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:69652:19: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!    }else if( xDel==SQLITE_DYNAMIC ){
#9 425.7 npm ERR!                    ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'valueFromExpr':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:70058:55: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!        sqlite3ValueSetStr(pVal, -1, zVal, SQLITE_UTF8, SQLITE_DYNAMIC);
#9 425.7 npm ERR!                                                        ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:70100:29: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!                           0, SQLITE_DYNAMIC);
#9 425.7 npm ERR!                              ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3VdbeExec':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:84132:38: warning: cast between incompatible function types from 'int (*)(u32,  void *, void *, void *)' {aka 'int (*)(unsigned int,  void *, void *, void *)'} to 'void (*)(void *, const char *)' [-Wcast-function-type]
#9 425.7 npm ERR!        void (*x)(void*,const char*) = (void(*)(void*,const char*))db->xTrace;
#9 425.7 npm ERR!                                       ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3VdbeExec':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:84132:38: warning: cast between incompatible function types from 'int (*)(u32,  void *, void *, void *)' {aka 'int (*)(unsigned int,  void *, void *, void *)'} to 'void (*)(void *, const char *)' [-Wcast-function-type]
#9 425.7 npm ERR!        void (*x)(void*,const char*) = (void(*)(void*,const char*))db->xTrace;
#9 425.7 npm ERR!                                       ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'renameTableFunc':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:94798:44: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!      sqlite3_result_text(context, zRet, -1, SQLITE_DYNAMIC);
#9 425.7 npm ERR!                                             ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'renameParentFunc':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:94862:45: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!    sqlite3_result_text(context, zResult, -1, SQLITE_DYNAMIC);
#9 425.7 npm ERR!                                              ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'renameTriggerFunc':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:94939:44: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!      sqlite3_result_text(context, zRet, -1, SQLITE_DYNAMIC);
#9 425.7 npm ERR!                                             ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'renameTableFunc':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:94798:44: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!      sqlite3_result_text(context, zRet, -1, SQLITE_DYNAMIC);
#9 425.7 npm ERR!                                             ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'renameParentFunc':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:94862:45: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!    sqlite3_result_text(context, zResult, -1, SQLITE_DYNAMIC);
#9 425.7 npm ERR!                                              ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'renameTriggerFunc':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:94939:44: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!      sqlite3_result_text(context, zRet, -1, SQLITE_DYNAMIC);
#9 425.7 npm ERR!                                             ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'printfFunc':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:104325:25: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!                          SQLITE_DYNAMIC);
#9 425.7 npm ERR!                          ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'printfFunc':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:104325:25: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!                          SQLITE_DYNAMIC);
#9 425.7 npm ERR!                          ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'generateColumnNames':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:115928:54: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!              sqlite3DbStrDup(db, pEList->a[i].zSpan), SQLITE_DYNAMIC);
#9 425.7 npm ERR!                                                       ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:115932:58: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!          sqlite3VdbeSetColName(v, i, COLNAME_NAME, zName, SQLITE_DYNAMIC);
#9 425.7 npm ERR!                                                           ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:115939:52: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!        sqlite3VdbeSetColName(v, i, COLNAME_NAME, z, SQLITE_DYNAMIC);
#9 425.7 npm ERR!                                                     ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'generateColumnNames':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:115928:54: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!              sqlite3DbStrDup(db, pEList->a[i].zSpan), SQLITE_DYNAMIC);
#9 425.7 npm ERR!                                                       ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:115932:58: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!          sqlite3VdbeSetColName(v, i, COLNAME_NAME, zName, SQLITE_DYNAMIC);
#9 425.7 npm ERR!                                                           ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:11909:27: warning: cast between incompatible function types from 'int (*)(void *)' to 'void (*)(void *)' [-Wcast-function-type]
#9 425.7 npm ERR!  #define SQLITE_DYNAMIC   ((sqlite3_destructor_type)sqlite3MallocSize)
#9 425.7 npm ERR!                            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:115939:52: note: in expansion of macro 'SQLITE_DYNAMIC'
#9 425.7 npm ERR!        sqlite3VdbeSetColName(v, i, COLNAME_NAME, z, SQLITE_DYNAMIC);
#9 425.7 npm ERR!                                                     ^~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3_trace':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:139629:16: warning: cast between incompatible function types from 'void (*)(void *, const char *)' to 'int (*)(u32,  void *, void *, void *)' {aka 'int (*)(unsigned int,  void *, void *, void *)'} [-Wcast-function-type]
#9 425.7 npm ERR!    db->xTrace = (int(*)(u32,void*,void*,void*))xTrace;
#9 425.7 npm ERR!                 ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3_trace':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:139629:16: warning: cast between incompatible function types from 'void (*)(void *, const char *)' to 'int (*)(u32,  void *, void *, void *)' {aka 'int (*)(unsigned int,  void *, void *, void *)'} [-Wcast-function-type]
#9 425.7 npm ERR!    db->xTrace = (int(*)(u32,void*,void*,void*))xTrace;
#9 425.7 npm ERR!                 ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3VXPrintf':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:25065:19: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!          flag_long = sizeof(char*)==sizeof(long int);
#9 425.7 npm ERR!          ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:25067:7: note: here
#9 425.7 npm ERR!        case etORDINAL:
#9 425.7 npm ERR!        ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3GetToken':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:137154:9: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!        if( !sqlite3Isdigit(z[1]) )
#9 425.7 npm ERR!          ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:137163:5: note: here
#9 425.7 npm ERR!      case CC_DIGIT: {
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:137253:9: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!        if( z[1]=='\'' ){
#9 425.7 npm ERR!          ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:137267:5: note: here
#9 425.7 npm ERR!      case CC_ID: {
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3ExprCodeTarget':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:93034:25: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!      case TK_AGG_COLUMN: {
#9 425.7 npm ERR!                          ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:93047:5: note: here
#9 425.7 npm ERR!      case TK_COLUMN: {
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'resolveP2Values':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:71026:13: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!            if( pOp->p2!=0 ) p->readOnly = 0;
#9 425.7 npm ERR!              ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:71029:9: note: here
#9 425.7 npm ERR!          case OP_AutoCommit:
#9 425.7 npm ERR!          ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'fkActionTrigger':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:107247:11: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!          if( !pChanges ){
#9 425.7 npm ERR!            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:107251:7: note: here
#9 425.7 npm ERR!        default:
#9 425.7 npm ERR!        ^~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3GenerateConstraintChecks':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:108736:17: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!          onError = OE_Abort;
#9 425.7 npm ERR!                  ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:108739:7: note: here
#9 425.7 npm ERR!        case OE_Rollback:
#9 425.7 npm ERR!        ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3VdbeExec':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:78202:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!    if( (pIn3->flags & MEM_Null)==0 ) break;
#9 425.7 npm ERR!      ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:78236:1: note: here
#9 425.7 npm ERR!  case OP_Halt: {
#9 425.7 npm ERR!  ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:78367:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!    if( pOp->p1>db->aLimit[SQLITE_LIMIT_LENGTH] ){
#9 425.7 npm ERR!      ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:78386:1: note: here
#9 425.7 npm ERR!  case OP_String: {          /* out2 */
#9 425.7 npm ERR!  ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:78941:15: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!    pOp->opcode = OP_Function;
#9 425.7 npm ERR!                ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:78944:1: note: here
#9 425.7 npm ERR!  case OP_Function: {
#9 425.7 npm ERR!  ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:82140:38: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!    p->aCounter[SQLITE_STMTSTATUS_SORT]++;
#9 425.7 npm ERR!    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:82155:1: note: here
#9 425.7 npm ERR!  case OP_Rewind: {        /* jump */
#9 425.7 npm ERR!  ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:83361:15: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!    pOp->opcode = OP_AggStep;
#9 425.7 npm ERR!                ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:83364:1: note: here
#9 425.7 npm ERR!  case OP_AggStep: {
#9 425.7 npm ERR!  ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'fts3SnippetFunc':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:146657:22: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!      case 6: nToken = sqlite3_value_int(apVal[5]);
#9 425.7 npm ERR!                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:146658:5: note: here
#9 425.7 npm ERR!      case 5: iCol = sqlite3_value_int(apVal[4]);
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:146658:20: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!      case 5: iCol = sqlite3_value_int(apVal[4]);
#9 425.7 npm ERR!                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:146659:5: note: here
#9 425.7 npm ERR!      case 4: zEllipsis = (const char*)sqlite3_value_text(apVal[3]);
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:146659:38: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!      case 4: zEllipsis = (const char*)sqlite3_value_text(apVal[3]);
#9 425.7 npm ERR!                                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:146660:5: note: here
#9 425.7 npm ERR!      case 3: zEnd = (const char*)sqlite3_value_text(apVal[2]);
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:146660:33: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!      case 3: zEnd = (const char*)sqlite3_value_text(apVal[2]);
#9 425.7 npm ERR!                                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:146661:5: note: here
#9 425.7 npm ERR!      case 2: zStart = (const char*)sqlite3_value_text(apVal[1]);
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'rtreeNonleafConstraint':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:162310:9: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!        if( p->op!=RTREE_EQ ) break;  /* RTREE_LE and RTREE_LT end here */
#9 425.7 npm ERR!          ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:162313:5: note: here
#9 425.7 npm ERR!      default: /* RTREE_GT or RTREE_GE,  or fallthrough of RTREE_EQ */
#9 425.7 npm ERR!      ^~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'jsonRenderNode':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:176768:9: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!        if( pNode->jnFlags & JNODE_RAW ){
#9 425.7 npm ERR!          ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:176774:5: note: here
#9 425.7 npm ERR!      case JSON_REAL:
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'jsonReturn':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:176891:7: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!        int_as_real: /* fall through to real */;
#9 425.7 npm ERR!        ^~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:176893:5: note: here
#9 425.7 npm ERR!      case JSON_REAL: {
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'jsonEachColumn':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:178258:9: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!        if( p->bRecursive ){
#9 425.7 npm ERR!          ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:178268:5: note: here
#9 425.7 npm ERR!      case JEACH_ROOT: {
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3VXPrintf':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:25065:19: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!          flag_long = sizeof(char*)==sizeof(long int);
#9 425.7 npm ERR!          ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:25067:7: note: here
#9 425.7 npm ERR!        case etORDINAL:
#9 425.7 npm ERR!        ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3GetToken':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:137154:9: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!        if( !sqlite3Isdigit(z[1]) )
#9 425.7 npm ERR!          ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:137163:5: note: here
#9 425.7 npm ERR!      case CC_DIGIT: {
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:137253:9: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!        if( z[1]=='\'' ){
#9 425.7 npm ERR!          ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:137267:5: note: here
#9 425.7 npm ERR!      case CC_ID: {
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3ExprCodeTarget':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:93034:25: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!      case TK_AGG_COLUMN: {
#9 425.7 npm ERR!                          ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:93047:5: note: here
#9 425.7 npm ERR!      case TK_COLUMN: {
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'resolveP2Values':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:71026:13: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!            if( pOp->p2!=0 ) p->readOnly = 0;
#9 425.7 npm ERR!              ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:71029:9: note: here
#9 425.7 npm ERR!          case OP_AutoCommit:
#9 425.7 npm ERR!          ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'fkActionTrigger':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:107247:11: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!          if( !pChanges ){
#9 425.7 npm ERR!            ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:107251:7: note: here
#9 425.7 npm ERR!        default:
#9 425.7 npm ERR!        ^~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3GenerateConstraintChecks':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:108736:17: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!          onError = OE_Abort;
#9 425.7 npm ERR!                  ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:108739:7: note: here
#9 425.7 npm ERR!        case OE_Rollback:
#9 425.7 npm ERR!        ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3VdbeExec':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:78202:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!    if( (pIn3->flags & MEM_Null)==0 ) break;
#9 425.7 npm ERR!      ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:78236:1: note: here
#9 425.7 npm ERR!  case OP_Halt: {
#9 425.7 npm ERR!  ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:78367:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!    if( pOp->p1>db->aLimit[SQLITE_LIMIT_LENGTH] ){
#9 425.7 npm ERR!      ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:78386:1: note: here
#9 425.7 npm ERR!  case OP_String: {          /* out2 */
#9 425.7 npm ERR!  ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:78941:15: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!    pOp->opcode = OP_Function;
#9 425.7 npm ERR!                ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:78944:1: note: here
#9 425.7 npm ERR!  case OP_Function: {
#9 425.7 npm ERR!  ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:82140:38: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!    p->aCounter[SQLITE_STMTSTATUS_SORT]++;
#9 425.7 npm ERR!    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:82155:1: note: here
#9 425.7 npm ERR!  case OP_Rewind: {        /* jump */
#9 425.7 npm ERR!  ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:83361:15: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!    pOp->opcode = OP_AggStep;
#9 425.7 npm ERR!                ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:83364:1: note: here
#9 425.7 npm ERR!  case OP_AggStep: {
#9 425.7 npm ERR!  ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'fts3SnippetFunc':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:146657:22: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!      case 6: nToken = sqlite3_value_int(apVal[5]);
#9 425.7 npm ERR!                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:146658:5: note: here
#9 425.7 npm ERR!      case 5: iCol = sqlite3_value_int(apVal[4]);
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:146658:20: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!      case 5: iCol = sqlite3_value_int(apVal[4]);
#9 425.7 npm ERR!                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:146659:5: note: here
#9 425.7 npm ERR!      case 4: zEllipsis = (const char*)sqlite3_value_text(apVal[3]);
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:146659:38: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!      case 4: zEllipsis = (const char*)sqlite3_value_text(apVal[3]);
#9 425.7 npm ERR!                                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:146660:5: note: here
#9 425.7 npm ERR!      case 3: zEnd = (const char*)sqlite3_value_text(apVal[2]);
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:146660:33: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!      case 3: zEnd = (const char*)sqlite3_value_text(apVal[2]);
#9 425.7 npm ERR!                                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:146661:5: note: here
#9 425.7 npm ERR!      case 2: zStart = (const char*)sqlite3_value_text(apVal[1]);
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'rtreeNonleafConstraint':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:162310:9: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!        if( p->op!=RTREE_EQ ) break;  /* RTREE_LE and RTREE_LT end here */
#9 425.7 npm ERR!          ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:162313:5: note: here
#9 425.7 npm ERR!      default: /* RTREE_GT or RTREE_GE,  or fallthrough of RTREE_EQ */
#9 425.7 npm ERR!      ^~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'jsonRenderNode':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:176768:9: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!        if( pNode->jnFlags & JNODE_RAW ){
#9 425.7 npm ERR!          ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:176774:5: note: here
#9 425.7 npm ERR!      case JSON_REAL:
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'jsonReturn':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:176891:7: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!        int_as_real: /* fall through to real */;
#9 425.7 npm ERR!        ^~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:176893:5: note: here
#9 425.7 npm ERR!      case JSON_REAL: {
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'jsonEachColumn':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:178258:9: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 425.7 npm ERR!        if( p->bRecursive ){
#9 425.7 npm ERR!          ^
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:178268:5: note: here
#9 425.7 npm ERR!      case JEACH_ROOT: {
#9 425.7 npm ERR!      ^~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'checkPtrmap':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:67133:5: warning: 'iPtrmapParent' may be used uninitialized in this function [-Wmaybe-uninitialized]
#9 425.7 npm ERR!      checkAppendMsg(pCheck,
#9 425.7 npm ERR!      ^~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR!        "Bad ptr map entry key=%d expected=(%d,%d) got=(%d,%d)",
#9 425.7 npm ERR!        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR!        iChild, eType, iParent, ePtrmapType, iPtrmapParent);
#9 425.7 npm ERR!        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:67133:5: warning: 'ePtrmapType' may be used uninitialized in this function [-Wmaybe-uninitialized]
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'checkPtrmap':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:67133:5: warning: 'iPtrmapParent' may be used uninitialized in this function [-Wmaybe-uninitialized]
#9 425.7 npm ERR!      checkAppendMsg(pCheck,
#9 425.7 npm ERR!      ^~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR!        "Bad ptr map entry key=%d expected=(%d,%d) got=(%d,%d)",
#9 425.7 npm ERR!        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR!        iChild, eType, iParent, ePtrmapType, iPtrmapParent);
#9 425.7 npm ERR!        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:67133:5: warning: 'ePtrmapType' may be used uninitialized in this function [-Wmaybe-uninitialized]
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3Fts5IndexQuery':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:191826:5: warning: 'memcpy' pointer overflow between offset 0 and size [-1, 9223372036854775807] [-Warray-bounds]
#9 425.7 npm ERR!      memcpy(&buf.p[1], pToken, nToken);
#9 425.7 npm ERR!      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:191826:5: warning: 'memcpy' specified size 18446744073709551615 exceeds maximum object size 9223372036854775807 [-Wstringop-overflow=]
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c: In function 'sqlite3Fts5IndexQuery':
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:191826:5: warning: 'memcpy' pointer overflow between offset 0 and size [-1, 9223372036854775807] [-Warray-bounds]
#9 425.7 npm ERR!      memcpy(&buf.p[1], pToken, nToken);
#9 425.7 npm ERR!      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR! Release/obj/gen/sqlite-autoconf-3150000/sqlite3.c:191826:5: warning: 'memcpy' specified size 18446744073709551615 exceeds maximum object size 9223372036854775807 [-Wstringop-overflow=]
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:192,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan_maybe_43_inl.h: In function 'Nan::Maybe<bool> Nan::ForceSet(v8::Local<v8::Object>, v8::Local<v8::Value>, v8::Local<v8::Value>, v8::PropertyAttribute)':
#9 425.7 npm ERR! ../node_modules/nan/nan_maybe_43_inl.h:112:15: error: 'class v8::Object' has no member named 'ForceSet'
#9 425.7 npm ERR!    return obj->ForceSet(isolate->GetCurrentContext(), key, value, attribs);
#9 425.7 npm ERR!                ^~~~~~~~
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan_converters.h:67,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:197,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan_converters_43_inl.h: In static member function 'static Nan::imp::ToFactoryBase<v8::Boolean>::return_t Nan::imp::ToFactory<v8::Boolean>::convert(v8::Local<v8::Value>)':
#9 425.7 npm ERR! ../node_modules/nan/nan_converters_43_inl.h:18:69: error: no matching function for call to 'v8::Value::ToBoolean(v8::Local<v8::Context>)'
#9 425.7 npm ERR!        val->To ## TYPE(v8::Isolate::GetCurrent()->GetCurrentContext())          \
#9 425.7 npm ERR!                                                                      ^
#9 425.7 npm ERR! ../node_modules/nan/nan_converters_43_inl.h:22:1: note: in expansion of macro 'X'
#9 425.7 npm ERR!  X(Boolean)
#9 425.7 npm ERR!  ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3086:18: note: candidate: 'v8::Local<v8::Boolean> v8::Value::ToBoolean(v8::Isolate*) const'
#9 425.7 npm ERR!    Local<Boolean> ToBoolean(Isolate* isolate) const;
#9 425.7 npm ERR!                   ^~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3086:18: note:   no known conversion for argument 1 from 'v8::Local<v8::Context>' to 'v8::Isolate*'
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan_converters.h:67,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:197,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan_converters_43_inl.h: In static member function 'static Nan::imp::ValueFactoryBase<bool>::return_t Nan::imp::ToFactory<bool>::convert(v8::Local<v8::Value>)':
#9 425.7 npm ERR! ../node_modules/nan/nan_converters_43_inl.h:37:57: error: no matching function for call to 'v8::Value::BooleanValue(v8::Local<v8::Context>)'
#9 425.7 npm ERR!    return val->NAME ## Value(isolate->GetCurrentContext());                     \
#9 425.7 npm ERR!                                                          ^
#9 425.7 npm ERR! ../node_modules/nan/nan_converters_43_inl.h:40:1: note: in expansion of macro 'X'
#9 425.7 npm ERR!  X(bool, Boolean)
#9 425.7 npm ERR!  ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3096:8: note: candidate: 'bool v8::Value::BooleanValue(v8::Isolate*) const'
#9 425.7 npm ERR!    bool BooleanValue(Isolate* isolate) const;
#9 425.7 npm ERR!         ^~~~~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3096:8: note:   no known conversion for argument 1 from 'v8::Local<v8::Context>' to 'v8::Isolate*'
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan_new.h:189,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:198,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan_implementation_12_inl.h: In static member function 'static Nan::imp::FactoryBase<v8::Function>::return_t Nan::imp::Factory<v8::Function>::New(Nan::FunctionCallback, v8::Local<v8::Value>)':
#9 425.7 npm ERR! ../node_modules/nan/nan_implementation_12_inl.h:105:32: error: no matching function for call to 'v8::Function::New(v8::Isolate*&, void (&)(const v8::FunctionCallbackInfo<v8::Value>&), v8::Local<v8::Object>&)'
#9 425.7 npm ERR!                            , obj));
#9 425.7 npm ERR!                                 ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4753:31: note: candidate: 'static v8::MaybeLocal<v8::Function> v8::Function::New(v8::Local<v8::Context>, v8::FunctionCallback, v8::Local<v8::Value>, int, v8::ConstructorBehavior, v8::SideEffectType)'
#9 425.7 npm ERR!    static MaybeLocal<Function> New(
#9 425.7 npm ERR!                                ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4753:31: note:   no known conversion for argument 1 from 'v8::Isolate*' to 'v8::Local<v8::Context>'
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan_new.h:189,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:198,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan_implementation_12_inl.h: In static member function 'static Nan::imp::FactoryBase<v8::StringObject>::return_t Nan::imp::Factory<v8::StringObject>::New(v8::Local<v8::String>)':
#9 425.7 npm ERR! ../node_modules/nan/nan_implementation_12_inl.h:337:37: error: no matching function for call to 'v8::StringObject::New(v8::Local<v8::String>&)'
#9 425.7 npm ERR!    return v8::StringObject::New(value).As<v8::StringObject>();
#9 425.7 npm ERR!                                      ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:6196:23: note: candidate: 'static v8::Local<v8::Value> v8::StringObject::New(v8::Isolate*, v8::Local<v8::String>)'
#9 425.7 npm ERR!    static Local<Value> New(Isolate* isolate, Local<String> value);
#9 425.7 npm ERR!                        ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:6196:23: note:   candidate expects 2 arguments, 1 provided
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan_new.h:189,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:198,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan_implementation_12_inl.h:337:58: error: expected primary-expression before '>' token
#9 425.7 npm ERR!    return v8::StringObject::New(value).As<v8::StringObject>();
#9 425.7 npm ERR!                                                           ^
#9 425.7 npm ERR! ../node_modules/nan/nan_implementation_12_inl.h:337:60: error: expected primary-expression before ')' token
#9 425.7 npm ERR!    return v8::StringObject::New(value).As<v8::StringObject>();
#9 425.7 npm ERR!                                                             ^
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In function 'v8::Local<v8::Value> Nan::MakeCallback(v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:835:60: warning: 'v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)' is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
#9 425.7 npm ERR!          v8::Isolate::GetCurrent(), target, func, argc, argv);
#9 425.7 npm ERR!                                                             ^
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:195:50: note: declared here
#9 425.7 npm ERR!                  NODE_EXTERN v8::Local<v8::Value> MakeCallback(
#9 425.7 npm ERR!                                                   ^~~~~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:112:42: note: in definition of macro 'NODE_DEPRECATED'
#9 425.7 npm ERR!      __attribute__((deprecated(message))) declarator
#9 425.7 npm ERR!                                           ^~~~~~~~~~
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h:835:60: warning: 'v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)' is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
#9 425.7 npm ERR!          v8::Isolate::GetCurrent(), target, func, argc, argv);
#9 425.7 npm ERR!                                                             ^
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:195:50: note: declared here
#9 425.7 npm ERR!                  NODE_EXTERN v8::Local<v8::Value> MakeCallback(
#9 425.7 npm ERR!                                                   ^~~~~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:112:42: note: in definition of macro 'NODE_DEPRECATED'
#9 425.7 npm ERR!      __attribute__((deprecated(message))) declarator
#9 425.7 npm ERR!                                           ^~~~~~~~~~
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In function 'v8::Local<v8::Value> Nan::MakeCallback(v8::Local<v8::Object>, v8::Local<v8::String>, int, v8::Local<v8::Value>*)':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:850:62: warning: 'v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::String>, int, v8::Local<v8::Value>*)' is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
#9 425.7 npm ERR!          v8::Isolate::GetCurrent(), target, symbol, argc, argv);
#9 425.7 npm ERR!                                                               ^
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:188:50: note: declared here
#9 425.7 npm ERR!                  NODE_EXTERN v8::Local<v8::Value> MakeCallback(
#9 425.7 npm ERR!                                                   ^~~~~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:112:42: note: in definition of macro 'NODE_DEPRECATED'
#9 425.7 npm ERR!      __attribute__((deprecated(message))) declarator
#9 425.7 npm ERR!                                           ^~~~~~~~~~
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h:850:62: warning: 'v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::String>, int, v8::Local<v8::Value>*)' is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
#9 425.7 npm ERR!          v8::Isolate::GetCurrent(), target, symbol, argc, argv);
#9 425.7 npm ERR!                                                               ^
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:188:50: note: declared here
#9 425.7 npm ERR!                  NODE_EXTERN v8::Local<v8::Value> MakeCallback(
#9 425.7 npm ERR!                                                   ^~~~~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:112:42: note: in definition of macro 'NODE_DEPRECATED'
#9 425.7 npm ERR!      __attribute__((deprecated(message))) declarator
#9 425.7 npm ERR!                                           ^~~~~~~~~~
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In function 'v8::Local<v8::Value> Nan::MakeCallback(v8::Local<v8::Object>, const char*, int, v8::Local<v8::Value>*)':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:865:62: warning: 'v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, const char*, int, v8::Local<v8::Value>*)' is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
#9 425.7 npm ERR!          v8::Isolate::GetCurrent(), target, method, argc, argv);
#9 425.7 npm ERR!                                                               ^
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:181:50: note: declared here
#9 425.7 npm ERR!                  NODE_EXTERN v8::Local<v8::Value> MakeCallback(
#9 425.7 npm ERR!                                                   ^~~~~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:112:42: note: in definition of macro 'NODE_DEPRECATED'
#9 425.7 npm ERR!      __attribute__((deprecated(message))) declarator
#9 425.7 npm ERR!                                           ^~~~~~~~~~
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h:865:62: warning: 'v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, const char*, int, v8::Local<v8::Value>*)' is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
#9 425.7 npm ERR!          v8::Isolate::GetCurrent(), target, method, argc, argv);
#9 425.7 npm ERR!                                                               ^
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:181:50: note: declared here
#9 425.7 npm ERR!                  NODE_EXTERN v8::Local<v8::Value> MakeCallback(
#9 425.7 npm ERR!                                                   ^~~~~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:112:42: note: in definition of macro 'NODE_DEPRECATED'
#9 425.7 npm ERR!      __attribute__((deprecated(message))) declarator
#9 425.7 npm ERR!                                           ^~~~~~~~~~
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In constructor 'Nan::Utf8String::Utf8String(v8::Local<v8::Value>)':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:911:53: error: no matching function for call to 'v8::Value::ToString()'
#9 425.7 npm ERR!        v8::Local<v8::String> string = from->ToString();
#9 425.7 npm ERR!                                                      ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3048:44: note: candidate: 'v8::MaybeLocal<v8::String> v8::Value::ToString(v8::Local<v8::Context>) const'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT MaybeLocal<String> ToString(
#9 425.7 npm ERR!                                             ^~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3048:44: note:   candidate expects 1 argument, 0 provided
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h:921:74: error: no matching function for call to 'v8::String::WriteUtf8(char*&, int, int, const int&)'
#9 425.7 npm ERR!          length_ = string->WriteUtf8(str_, static_cast<int>(len), 0, flags);
#9 425.7 npm ERR!                                                                           ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3277:7: note: candidate: 'int v8::String::WriteUtf8(v8::Isolate*, char*, int, int*, int) const'
#9 425.7 npm ERR!    int WriteUtf8(Isolate* isolate, char* buffer, int length = -1,
#9 425.7 npm ERR!        ^~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3277:7: note:   no known conversion for argument 1 from 'char*' to 'v8::Isolate*'
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In member function 'v8::Local<v8::Value> Nan::Callback::Call_(v8::Isolate*, v8::Local<v8::Object>, int, v8::Local<v8::Value>*) const':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:1479:5: warning: 'v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)' is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
#9 425.7 npm ERR!      ));
#9 425.7 npm ERR!      ^
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:195:50: note: declared here
#9 425.7 npm ERR!                  NODE_EXTERN v8::Local<v8::Value> MakeCallback(
#9 425.7 npm ERR!                                                   ^~~~~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:112:42: note: in definition of macro 'NODE_DEPRECATED'
#9 425.7 npm ERR!      __attribute__((deprecated(message))) declarator
#9 425.7 npm ERR!                                           ^~~~~~~~~~
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h:1479:5: warning: 'v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)' is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
#9 425.7 npm ERR!      ));
#9 425.7 npm ERR!      ^
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:195:50: note: declared here
#9 425.7 npm ERR!                  NODE_EXTERN v8::Local<v8::Value> MakeCallback(
#9 425.7 npm ERR!                                                   ^~~~~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:112:42: note: in definition of macro 'NODE_DEPRECATED'
#9 425.7 npm ERR!      __attribute__((deprecated(message))) declarator
#9 425.7 npm ERR!                                           ^~~~~~~~~~
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In member function 'void Nan::AsyncWorker::SaveToPersistent(const char*, const v8::Local<v8::Value>&)':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:1533:64: error: no matching function for call to 'v8::Object::Set(v8::Local<v8::String>, const v8::Local<v8::Value>&)'
#9 425.7 npm ERR!      New(persistentHandle)->Set(New(key).ToLocalChecked(), value);
#9 425.7 npm ERR!                                                                 ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3961:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, v8::Local<v8::Value>, v8::Local<v8::Value>)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
#9 425.7 npm ERR!                                      ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3961:37: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3964:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, uint32_t, v8::Local<v8::Value>)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
#9 425.7 npm ERR!                                      ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3964:37: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In member function 'void Nan::AsyncWorker::SaveToPersistent(const v8::Local<v8::String>&, const v8::Local<v8::Value>&)':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:1539:42: error: no matching function for call to 'v8::Object::Set(const v8::Local<v8::String>&, const v8::Local<v8::Value>&)'
#9 425.7 npm ERR!      New(persistentHandle)->Set(key, value);
#9 425.7 npm ERR!                                           ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3961:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, v8::Local<v8::Value>, v8::Local<v8::Value>)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
#9 425.7 npm ERR!                                      ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3961:37: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3964:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, uint32_t, v8::Local<v8::Value>)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
#9 425.7 npm ERR!                                      ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3964:37: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In member function 'void Nan::AsyncWorker::SaveToPersistent(uint32_t, const v8::Local<v8::Value>&)':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:1545:44: error: no matching function for call to 'v8::Object::Set(uint32_t&, const v8::Local<v8::Value>&)'
#9 425.7 npm ERR!      New(persistentHandle)->Set(index, value);
#9 425.7 npm ERR!                                             ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3961:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, v8::Local<v8::Value>, v8::Local<v8::Value>)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
#9 425.7 npm ERR!                                      ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3961:37: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3964:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, uint32_t, v8::Local<v8::Value>)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
#9 425.7 npm ERR!                                      ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3964:37: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In member function 'v8::Local<v8::Value> Nan::AsyncWorker::GetFromPersistent(const char*) const':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:1551:61: error: no matching function for call to 'v8::Object::Get(v8::Local<v8::String>)'
#9 425.7 npm ERR!          New(persistentHandle)->Get(New(key).ToLocalChecked()));
#9 425.7 npm ERR!                                                              ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4007:43: note: candidate: 'v8::MaybeLocal<v8::Value> v8::Object::Get(v8::Local<v8::Context>, v8::Local<v8::Value>)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
#9 425.7 npm ERR!                                            ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4007:43: note:   candidate expects 2 arguments, 1 provided
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4010:43: note: candidate: 'v8::MaybeLocal<v8::Value> v8::Object::Get(v8::Local<v8::Context>, uint32_t)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
#9 425.7 npm ERR!                                            ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4010:43: note:   candidate expects 2 arguments, 1 provided
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In member function 'v8::Local<v8::Value> Nan::AsyncWorker::GetFromPersistent(const v8::Local<v8::String>&) const':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:1557:55: error: no matching function for call to 'v8::Object::Get(const v8::Local<v8::String>&)'
#9 425.7 npm ERR!      return scope.Escape(New(persistentHandle)->Get(key));
#9 425.7 npm ERR!                                                        ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4007:43: note: candidate: 'v8::MaybeLocal<v8::Value> v8::Object::Get(v8::Local<v8::Context>, v8::Local<v8::Value>)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
#9 425.7 npm ERR!                                            ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4007:43: note:   candidate expects 2 arguments, 1 provided
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4010:43: note: candidate: 'v8::MaybeLocal<v8::Value> v8::Object::Get(v8::Local<v8::Context>, uint32_t)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
#9 425.7 npm ERR!                                            ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4010:43: note:   candidate expects 2 arguments, 1 provided
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In member function 'v8::Local<v8::Value> Nan::AsyncWorker::GetFromPersistent(uint32_t) const':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:1562:57: error: no matching function for call to 'v8::Object::Get(uint32_t&)'
#9 425.7 npm ERR!      return scope.Escape(New(persistentHandle)->Get(index));
#9 425.7 npm ERR!                                                          ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4007:43: note: candidate: 'v8::MaybeLocal<v8::Value> v8::Object::Get(v8::Local<v8::Context>, v8::Local<v8::Value>)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
#9 425.7 npm ERR!                                            ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4007:43: note:   candidate expects 2 arguments, 1 provided
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4010:43: note: candidate: 'v8::MaybeLocal<v8::Value> v8::Object::Get(v8::Local<v8::Context>, uint32_t)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
#9 425.7 npm ERR!                                            ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4010:43: note:   candidate expects 2 arguments, 1 provided
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In function 'void Nan::AsyncQueueWorker(Nan::AsyncWorker*)':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:1732:62: warning: cast between incompatible function types from 'void (*)(uv_work_t*)' {aka 'void (*)(uv_work_s*)'} to 'uv_after_work_cb' {aka 'void (*)(uv_work_s*, int)'} [-Wcast-function-type]
#9 425.7 npm ERR!      , reinterpret_cast<uv_after_work_cb>(AsyncExecuteComplete)
#9 425.7 npm ERR!                                                               ^
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:2222,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan_object_wrap.h: In member function 'void Nan::ObjectWrap::MakeWeak()':
#9 425.7 npm ERR! ../node_modules/nan/nan_object_wrap.h:67:18: error: 'class Nan::Persistent<v8::Object>' has no member named 'MarkIndependent'
#9 425.7 npm ERR!      persistent().MarkIndependent();
#9 425.7 npm ERR!                   ^~~~~~~~~~~~~~~
#9 425.7 npm ERR! In file included from ../src/database.cc:3:
#9 425.7 npm ERR! ../src/database.cc: In member function 'void node_sqlite3::Database::Process()':
#9 425.7 npm ERR! ../src/macros.h:91:13: error: no matching function for call to 'v8::String::Concat(v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!              ),                                                                 \
#9 425.7 npm ERR!              ^
#9 425.7 npm ERR! ../src/database.cc:40:9: note: in expansion of macro 'EXCEPTION'
#9 425.7 npm ERR!          EXCEPTION(Nan::New("Database handle is closed").ToLocalChecked(), SQLITE_MISUSE, exception);
#9 425.7 npm ERR!          ^~~~~~~~~
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note: candidate: 'static v8::Local<v8::String> v8::String::Concat(v8::Isolate*, v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!    static Local<String> Concat(Isolate* isolate, Local<String> left,
#9 425.7 npm ERR!                         ^~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! In file included from ../src/database.cc:3:
#9 425.7 npm ERR! ../src/database.cc: In member function 'void node_sqlite3::Database::Schedule(node_sqlite3::Database::Work_Callback, node_sqlite3::Database::Baton*, bool)':
#9 425.7 npm ERR! ../src/macros.h:91:13: error: no matching function for call to 'v8::String::Concat(v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!              ),                                                                 \
#9 425.7 npm ERR!              ^
#9 425.7 npm ERR! ../src/database.cc:88:9: note: in expansion of macro 'EXCEPTION'
#9 425.7 npm ERR!          EXCEPTION(Nan::New("Database is closed").ToLocalChecked(), SQLITE_MISUSE, exception);
#9 425.7 npm ERR!          ^~~~~~~~~
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note: candidate: 'static v8::Local<v8::String> v8::String::Concat(v8::Isolate*, v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!    static Local<String> Concat(Isolate* isolate, Local<String> left,
#9 425.7 npm ERR!                         ^~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! ../src/database.cc: In static member function 'static void node_sqlite3::Database::Work_BeginOpen(node_sqlite3::Database::Baton*)':
#9 425.7 npm ERR! ../src/database.cc:145:55: warning: cast between incompatible function types from 'void (*)(uv_work_t*)' {aka 'void (*)(uv_work_s*)'} to 'uv_after_work_cb' {aka 'void (*)(uv_work_s*, int)'} [-Wcast-function-type]
#9 425.7 npm ERR!          &baton->request, Work_Open, (uv_after_work_cb)Work_AfterOpen);
#9 425.7 npm ERR!                                                        ^~~~~~~~~~~~~~
#9 425.7 npm ERR! ../src/database.cc:144:9: warning: unused variable 'status' [-Wunused-variable]
#9 425.7 npm ERR!      int status = uv_queue_work(uv_default_loop(),
#9 425.7 npm ERR!          ^~~~~~
#9 425.7 npm ERR! In file included from ../src/database.cc:3:
#9 425.7 npm ERR! ../src/database.cc: In static member function 'static void node_sqlite3::Database::Work_AfterOpen(uv_work_t*)':
#9 425.7 npm ERR! ../src/macros.h:91:13: error: no matching function for call to 'v8::String::Concat(v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!              ),                                                                 \
#9 425.7 npm ERR!              ^
#9 425.7 npm ERR! ../src/database.cc:179:9: note: in expansion of macro 'EXCEPTION'
#9 425.7 npm ERR!          EXCEPTION(Nan::New(baton->message.c_str()).ToLocalChecked(), baton->status, exception);
#9 425.7 npm ERR!          ^~~~~~~~~
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note: candidate: 'static v8::Local<v8::String> v8::String::Concat(v8::Isolate*, v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!    static Local<String> Concat(Isolate* isolate, Local<String> left,
#9 425.7 npm ERR!                         ^~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! ../src/database.cc: In static member function 'static void node_sqlite3::Database::Work_BeginClose(node_sqlite3::Database::Baton*)':
#9 425.7 npm ERR! ../src/database.cc:231:56: warning: cast between incompatible function types from 'void (*)(uv_work_t*)' {aka 'void (*)(uv_work_s*)'} to 'uv_after_work_cb' {aka 'void (*)(uv_work_s*, int)'} [-Wcast-function-type]
#9 425.7 npm ERR!          &baton->request, Work_Close, (uv_after_work_cb)Work_AfterClose);
#9 425.7 npm ERR!                                                         ^~~~~~~~~~~~~~~
#9 425.7 npm ERR! ../src/database.cc:230:9: warning: unused variable 'status' [-Wunused-variable]
#9 425.7 npm ERR!      int status = uv_queue_work(uv_default_loop(),
#9 425.7 npm ERR!          ^~~~~~
#9 425.7 npm ERR! In file included from ../src/database.cc:3:
#9 425.7 npm ERR! ../src/database.cc: In static member function 'static void node_sqlite3::Database::Work_AfterClose(uv_work_t*)':
#9 425.7 npm ERR! ../src/macros.h:91:13: error: no matching function for call to 'v8::String::Concat(v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!              ),                                                                 \
#9 425.7 npm ERR!              ^
#9 425.7 npm ERR! ../src/database.cc:259:9: note: in expansion of macro 'EXCEPTION'
#9 425.7 npm ERR!          EXCEPTION(Nan::New(baton->message.c_str()).ToLocalChecked(), baton->status, exception);
#9 425.7 npm ERR!          ^~~~~~~~~
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note: candidate: 'static v8::Local<v8::String> v8::String::Concat(v8::Isolate*, v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!    static Local<String> Concat(Isolate* isolate, Local<String> left,
#9 425.7 npm ERR!                         ^~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! ../src/database.cc: In static member function 'static Nan::NAN_METHOD_RETURN_TYPE node_sqlite3::Database::Configure(Nan::NAN_METHOD_ARGS_TYPE)':
#9 425.7 npm ERR! ../src/database.cc:351:9: error: no matching function for call to 'v8::String::Concat(v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!          )));
#9 425.7 npm ERR!          ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note: candidate: 'static v8::Local<v8::String> v8::String::Concat(v8::Isolate*, v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!    static Local<String> Concat(Isolate* isolate, Local<String> left,
#9 425.7 npm ERR!                         ^~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! ../src/database.cc:351:11: error: return-statement with a value, in function returning 'Nan::NAN_METHOD_RETURN_TYPE' {aka 'void'} [-fpermissive]
#9 425.7 npm ERR!          )));
#9 425.7 npm ERR!            ^
#9 425.7 npm ERR! ../src/database.cc: In static member function 'static void node_sqlite3::Database::Work_BeginExec(node_sqlite3::Database::Baton*)':
#9 425.7 npm ERR! ../src/database.cc:526:55: warning: cast between incompatible function types from 'void (*)(uv_work_t*)' {aka 'void (*)(uv_work_s*)'} to 'uv_after_work_cb' {aka 'void (*)(uv_work_s*, int)'} [-Wcast-function-type]
#9 425.7 npm ERR!          &baton->request, Work_Exec, (uv_after_work_cb)Work_AfterExec);
#9 425.7 npm ERR!                                                        ^~~~~~~~~~~~~~
#9 425.7 npm ERR! ../src/database.cc:525:9: warning: unused variable 'status' [-Wunused-variable]
#9 425.7 npm ERR!      int status = uv_queue_work(uv_default_loop(),
#9 425.7 npm ERR!          ^~~~~~
#9 425.7 npm ERR! In file included from ../src/database.cc:3:
#9 425.7 npm ERR! ../src/database.cc: In static member function 'static void node_sqlite3::Database::Work_AfterExec(uv_work_t*)':
#9 425.7 npm ERR! ../src/macros.h:91:13: error: no matching function for call to 'v8::String::Concat(v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!              ),                                                                 \
#9 425.7 npm ERR!              ^
#9 425.7 npm ERR! ../src/database.cc:557:9: note: in expansion of macro 'EXCEPTION'
#9 425.7 npm ERR!          EXCEPTION(Nan::New(baton->message.c_str()).ToLocalChecked(), baton->status, exception);
#9 425.7 npm ERR!          ^~~~~~~~~
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note: candidate: 'static v8::Local<v8::String> v8::String::Concat(v8::Isolate*, v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!    static Local<String> Concat(Isolate* isolate, Local<String> left,
#9 425.7 npm ERR!                         ^~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! ../src/database.cc: In static member function 'static void node_sqlite3::Database::Work_BeginLoadExtension(node_sqlite3::Database::Baton*)':
#9 425.7 npm ERR! ../src/database.cc:626:104: warning: cast between incompatible function types from 'void (*)(uv_work_t*)' {aka 'void (*)(uv_work_s*)'} to 'uv_after_work_cb' {aka 'void (*)(uv_work_s*, int)'} [-Wcast-function-type]
#9 425.7 npm ERR!          &baton->request, Work_LoadExtension, reinterpret_cast<uv_after_work_cb>(Work_AfterLoadExtension));
#9 425.7 npm ERR!                                                                                                         ^
#9 425.7 npm ERR! ../src/database.cc:625:9: warning: unused variable 'status' [-Wunused-variable]
#9 425.7 npm ERR!      int status = uv_queue_work(uv_default_loop(),
#9 425.7 npm ERR!          ^~~~~~
#9 425.7 npm ERR! In file included from ../src/database.cc:3:
#9 425.7 npm ERR! ../src/database.cc: In static member function 'static void node_sqlite3::Database::Work_AfterLoadExtension(uv_work_t*)':
#9 425.7 npm ERR! ../src/macros.h:91:13: error: no matching function for call to 'v8::String::Concat(v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!              ),                                                                 \
#9 425.7 npm ERR!              ^
#9 425.7 npm ERR! ../src/database.cc:659:9: note: in expansion of macro 'EXCEPTION'
#9 425.7 npm ERR!          EXCEPTION(Nan::New(baton->message.c_str()).ToLocalChecked(), baton->status, exception);
#9 425.7 npm ERR!          ^~~~~~~~~
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note: candidate: 'static v8::Local<v8::String> v8::String::Concat(v8::Isolate*, v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!    static Local<String> Concat(Isolate* isolate, Local<String> left,
#9 425.7 npm ERR!                         ^~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! In file included from ../src/database.h:12,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../src/async.h: In instantiation of 'Async<Item, Parent>::Async(Parent*, Async<Item, Parent>::Callback) [with Item = std::__cxx11::basic_string<char>; Parent = node_sqlite3::Database; Async<Item, Parent>::Callback = void (*)(node_sqlite3::Database*, std::__cxx11::basic_string<char>*)]':
#9 425.7 npm ERR! ../src/database.cc:391:59:   required from here
#9 425.7 npm ERR! ../src/async.h:29:52: warning: cast between incompatible function types from 'void (*)(uv_async_t*, int)' {aka 'void (*)(uv_async_s*, int)'} to 'uv_async_cb' {aka 'void (*)(uv_async_s*)'} [-Wcast-function-type]
#9 425.7 npm ERR!          uv_async_init(uv_default_loop(), &watcher, reinterpret_cast<uv_async_cb>(listener));
#9 425.7 npm ERR!                                                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR! ../src/async.h: In instantiation of 'Async<Item, Parent>::Async(Parent*, Async<Item, Parent>::Callback) [with Item = node_sqlite3::Database::ProfileInfo; Parent = node_sqlite3::Database; Async<Item, Parent>::Callback = void (*)(node_sqlite3::Database*, node_sqlite3::Database::ProfileInfo*)]':
#9 425.7 npm ERR! ../src/database.cc:429:65:   required from here
#9 425.7 npm ERR! ../src/async.h:29:52: warning: cast between incompatible function types from 'void (*)(uv_async_t*, int)' {aka 'void (*)(uv_async_s*, int)'} to 'uv_async_cb' {aka 'void (*)(uv_async_s*)'} [-Wcast-function-type]
#9 425.7 npm ERR! ../src/async.h: In instantiation of 'Async<Item, Parent>::Async(Parent*, Async<Item, Parent>::Callback) [with Item = node_sqlite3::Database::UpdateInfo; Parent = node_sqlite3::Database; Async<Item, Parent>::Callback = void (*)(node_sqlite3::Database*, node_sqlite3::Database::UpdateInfo*)]':
#9 425.7 npm ERR! ../src/database.cc:470:62:   required from here
#9 425.7 npm ERR! ../src/async.h:29:52: warning: cast between incompatible function types from 'void (*)(uv_async_t*, int)' {aka 'void (*)(uv_async_s*, int)'} to 'uv_async_cb' {aka 'void (*)(uv_async_s*)'} [-Wcast-function-type]
#9 425.7 npm ERR! make: *** [node_sqlite3.target.mk:133: Release/obj.target/node_sqlite3/src/database.o] Error 1
#9 425.7 npm ERR! gyp ERR! build error 
#9 425.7 npm ERR! gyp ERR! stack Error: `make` failed with exit code: 2
#9 425.7 npm ERR! gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:201:23)
#9 425.7 npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
#9 425.7 npm ERR! gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:293:12)
#9 425.7 npm ERR! gyp ERR! System Linux 3.10.0-1127.10.1.el7.x86_64
#9 425.7 npm ERR! gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--module=/usr/src/app/node_modules/sqlite3/lib/binding/node-v93-linux-x64/node_sqlite3.node" "--module_name=node_sqlite3" "--module_path=/usr/src/app/node_modules/sqlite3/lib/binding/node-v93-linux-x64"
#9 425.7 npm ERR! gyp ERR! cwd /usr/src/app/node_modules/sqlite3
#9 425.7 npm ERR! gyp ERR! node -v v16.20.2
#9 425.7 npm ERR! gyp ERR! node-gyp -v v9.1.0
#9 425.7 npm ERR! gyp ERR! not ok 
#9 425.7 npm ERR! node-pre-gyp ERR! build error 
#9 425.7 npm ERR! node-pre-gyp ERR! stack Error: Failed to execute '/usr/local/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/usr/src/app/node_modules/sqlite3/lib/binding/node-v93-linux-x64/node_sqlite3.node --module_name=node_sqlite3 --module_path=/usr/src/app/node_modules/sqlite3/lib/binding/node-v93-linux-x64' (1)
#9 425.7 npm ERR! node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/usr/src/app/node_modules/sqlite3/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
#9 425.7 npm ERR! node-pre-gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
#9 425.7 npm ERR! node-pre-gyp ERR! stack     at maybeClose (node:internal/child_process:1100:16)
#9 425.7 npm ERR! node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:304:5)
#9 425.7 npm ERR! node-pre-gyp ERR! System Linux 3.10.0-1127.10.1.el7.x86_64
#9 425.7 npm ERR! node-pre-gyp ERR! command "/usr/local/bin/node" "/usr/src/app/node_modules/sqlite3/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
#9 425.7 npm ERR! node-pre-gyp ERR! cwd /usr/src/app/node_modules/sqlite3
#9 425.7 npm ERR! node-pre-gyp ERR! node -v v16.20.2
#9 425.7 npm ERR! node-pre-gyp ERR! node-pre-gyp -v v0.6.38
#9 425.7 npm ERR! node-pre-gyp ERR! not ok 
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:192,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan_maybe_43_inl.h: In function 'Nan::Maybe<bool> Nan::ForceSet(v8::Local<v8::Object>, v8::Local<v8::Value>, v8::Local<v8::Value>, v8::PropertyAttribute)':
#9 425.7 npm ERR! ../node_modules/nan/nan_maybe_43_inl.h:112:15: error: 'class v8::Object' has no member named 'ForceSet'
#9 425.7 npm ERR!    return obj->ForceSet(isolate->GetCurrentContext(), key, value, attribs);
#9 425.7 npm ERR!                ^~~~~~~~
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan_converters.h:67,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:197,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan_converters_43_inl.h: In static member function 'static Nan::imp::ToFactoryBase<v8::Boolean>::return_t Nan::imp::ToFactory<v8::Boolean>::convert(v8::Local<v8::Value>)':
#9 425.7 npm ERR! ../node_modules/nan/nan_converters_43_inl.h:18:69: error: no matching function for call to 'v8::Value::ToBoolean(v8::Local<v8::Context>)'
#9 425.7 npm ERR!        val->To ## TYPE(v8::Isolate::GetCurrent()->GetCurrentContext())          \
#9 425.7 npm ERR!                                                                      ^
#9 425.7 npm ERR! ../node_modules/nan/nan_converters_43_inl.h:22:1: note: in expansion of macro 'X'
#9 425.7 npm ERR!  X(Boolean)
#9 425.7 npm ERR!  ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3086:18: note: candidate: 'v8::Local<v8::Boolean> v8::Value::ToBoolean(v8::Isolate*) const'
#9 425.7 npm ERR!    Local<Boolean> ToBoolean(Isolate* isolate) const;
#9 425.7 npm ERR!                   ^~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3086:18: note:   no known conversion for argument 1 from 'v8::Local<v8::Context>' to 'v8::Isolate*'
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan_converters.h:67,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:197,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan_converters_43_inl.h: In static member function 'static Nan::imp::ValueFactoryBase<bool>::return_t Nan::imp::ToFactory<bool>::convert(v8::Local<v8::Value>)':
#9 425.7 npm ERR! ../node_modules/nan/nan_converters_43_inl.h:37:57: error: no matching function for call to 'v8::Value::BooleanValue(v8::Local<v8::Context>)'
#9 425.7 npm ERR!    return val->NAME ## Value(isolate->GetCurrentContext());                     \
#9 425.7 npm ERR!                                                          ^
#9 425.7 npm ERR! ../node_modules/nan/nan_converters_43_inl.h:40:1: note: in expansion of macro 'X'
#9 425.7 npm ERR!  X(bool, Boolean)
#9 425.7 npm ERR!  ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3096:8: note: candidate: 'bool v8::Value::BooleanValue(v8::Isolate*) const'
#9 425.7 npm ERR!    bool BooleanValue(Isolate* isolate) const;
#9 425.7 npm ERR!         ^~~~~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3096:8: note:   no known conversion for argument 1 from 'v8::Local<v8::Context>' to 'v8::Isolate*'
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan_new.h:189,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:198,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan_implementation_12_inl.h: In static member function 'static Nan::imp::FactoryBase<v8::Function>::return_t Nan::imp::Factory<v8::Function>::New(Nan::FunctionCallback, v8::Local<v8::Value>)':
#9 425.7 npm ERR! ../node_modules/nan/nan_implementation_12_inl.h:105:32: error: no matching function for call to 'v8::Function::New(v8::Isolate*&, void (&)(const v8::FunctionCallbackInfo<v8::Value>&), v8::Local<v8::Object>&)'
#9 425.7 npm ERR!                            , obj));
#9 425.7 npm ERR!                                 ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4753:31: note: candidate: 'static v8::MaybeLocal<v8::Function> v8::Function::New(v8::Local<v8::Context>, v8::FunctionCallback, v8::Local<v8::Value>, int, v8::ConstructorBehavior, v8::SideEffectType)'
#9 425.7 npm ERR!    static MaybeLocal<Function> New(
#9 425.7 npm ERR!                                ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4753:31: note:   no known conversion for argument 1 from 'v8::Isolate*' to 'v8::Local<v8::Context>'
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan_new.h:189,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:198,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan_implementation_12_inl.h: In static member function 'static Nan::imp::FactoryBase<v8::StringObject>::return_t Nan::imp::Factory<v8::StringObject>::New(v8::Local<v8::String>)':
#9 425.7 npm ERR! ../node_modules/nan/nan_implementation_12_inl.h:337:37: error: no matching function for call to 'v8::StringObject::New(v8::Local<v8::String>&)'
#9 425.7 npm ERR!    return v8::StringObject::New(value).As<v8::StringObject>();
#9 425.7 npm ERR!                                      ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:6196:23: note: candidate: 'static v8::Local<v8::Value> v8::StringObject::New(v8::Isolate*, v8::Local<v8::String>)'
#9 425.7 npm ERR!    static Local<Value> New(Isolate* isolate, Local<String> value);
#9 425.7 npm ERR!                        ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:6196:23: note:   candidate expects 2 arguments, 1 provided
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan_new.h:189,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:198,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan_implementation_12_inl.h:337:58: error: expected primary-expression before '>' token
#9 425.7 npm ERR!    return v8::StringObject::New(value).As<v8::StringObject>();
#9 425.7 npm ERR!                                                           ^
#9 425.7 npm ERR! ../node_modules/nan/nan_implementation_12_inl.h:337:60: error: expected primary-expression before ')' token
#9 425.7 npm ERR!    return v8::StringObject::New(value).As<v8::StringObject>();
#9 425.7 npm ERR!                                                             ^
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In function 'v8::Local<v8::Value> Nan::MakeCallback(v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:835:60: warning: 'v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)' is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
#9 425.7 npm ERR!          v8::Isolate::GetCurrent(), target, func, argc, argv);
#9 425.7 npm ERR!                                                             ^
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:195:50: note: declared here
#9 425.7 npm ERR!                  NODE_EXTERN v8::Local<v8::Value> MakeCallback(
#9 425.7 npm ERR!                                                   ^~~~~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:112:42: note: in definition of macro 'NODE_DEPRECATED'
#9 425.7 npm ERR!      __attribute__((deprecated(message))) declarator
#9 425.7 npm ERR!                                           ^~~~~~~~~~
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h:835:60: warning: 'v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)' is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
#9 425.7 npm ERR!          v8::Isolate::GetCurrent(), target, func, argc, argv);
#9 425.7 npm ERR!                                                             ^
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:195:50: note: declared here
#9 425.7 npm ERR!                  NODE_EXTERN v8::Local<v8::Value> MakeCallback(
#9 425.7 npm ERR!                                                   ^~~~~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:112:42: note: in definition of macro 'NODE_DEPRECATED'
#9 425.7 npm ERR!      __attribute__((deprecated(message))) declarator
#9 425.7 npm ERR!                                           ^~~~~~~~~~
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In function 'v8::Local<v8::Value> Nan::MakeCallback(v8::Local<v8::Object>, v8::Local<v8::String>, int, v8::Local<v8::Value>*)':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:850:62: warning: 'v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::String>, int, v8::Local<v8::Value>*)' is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
#9 425.7 npm ERR!          v8::Isolate::GetCurrent(), target, symbol, argc, argv);
#9 425.7 npm ERR!                                                               ^
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:188:50: note: declared here
#9 425.7 npm ERR!                  NODE_EXTERN v8::Local<v8::Value> MakeCallback(
#9 425.7 npm ERR!                                                   ^~~~~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:112:42: note: in definition of macro 'NODE_DEPRECATED'
#9 425.7 npm ERR!      __attribute__((deprecated(message))) declarator
#9 425.7 npm ERR!                                           ^~~~~~~~~~
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h:850:62: warning: 'v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::String>, int, v8::Local<v8::Value>*)' is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
#9 425.7 npm ERR!          v8::Isolate::GetCurrent(), target, symbol, argc, argv);
#9 425.7 npm ERR!                                                               ^
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:188:50: note: declared here
#9 425.7 npm ERR!                  NODE_EXTERN v8::Local<v8::Value> MakeCallback(
#9 425.7 npm ERR!                                                   ^~~~~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:112:42: note: in definition of macro 'NODE_DEPRECATED'
#9 425.7 npm ERR!      __attribute__((deprecated(message))) declarator
#9 425.7 npm ERR!                                           ^~~~~~~~~~
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In function 'v8::Local<v8::Value> Nan::MakeCallback(v8::Local<v8::Object>, const char*, int, v8::Local<v8::Value>*)':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:865:62: warning: 'v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, const char*, int, v8::Local<v8::Value>*)' is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
#9 425.7 npm ERR!          v8::Isolate::GetCurrent(), target, method, argc, argv);
#9 425.7 npm ERR!                                                               ^
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:181:50: note: declared here
#9 425.7 npm ERR!                  NODE_EXTERN v8::Local<v8::Value> MakeCallback(
#9 425.7 npm ERR!                                                   ^~~~~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:112:42: note: in definition of macro 'NODE_DEPRECATED'
#9 425.7 npm ERR!      __attribute__((deprecated(message))) declarator
#9 425.7 npm ERR!                                           ^~~~~~~~~~
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h:865:62: warning: 'v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, const char*, int, v8::Local<v8::Value>*)' is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
#9 425.7 npm ERR!          v8::Isolate::GetCurrent(), target, method, argc, argv);
#9 425.7 npm ERR!                                                               ^
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:181:50: note: declared here
#9 425.7 npm ERR!                  NODE_EXTERN v8::Local<v8::Value> MakeCallback(
#9 425.7 npm ERR!                                                   ^~~~~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:112:42: note: in definition of macro 'NODE_DEPRECATED'
#9 425.7 npm ERR!      __attribute__((deprecated(message))) declarator
#9 425.7 npm ERR!                                           ^~~~~~~~~~
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In constructor 'Nan::Utf8String::Utf8String(v8::Local<v8::Value>)':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:911:53: error: no matching function for call to 'v8::Value::ToString()'
#9 425.7 npm ERR!        v8::Local<v8::String> string = from->ToString();
#9 425.7 npm ERR!                                                      ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3048:44: note: candidate: 'v8::MaybeLocal<v8::String> v8::Value::ToString(v8::Local<v8::Context>) const'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT MaybeLocal<String> ToString(
#9 425.7 npm ERR!                                             ^~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3048:44: note:   candidate expects 1 argument, 0 provided
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h:921:74: error: no matching function for call to 'v8::String::WriteUtf8(char*&, int, int, const int&)'
#9 425.7 npm ERR!          length_ = string->WriteUtf8(str_, static_cast<int>(len), 0, flags);
#9 425.7 npm ERR!                                                                           ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3277:7: note: candidate: 'int v8::String::WriteUtf8(v8::Isolate*, char*, int, int*, int) const'
#9 425.7 npm ERR!    int WriteUtf8(Isolate* isolate, char* buffer, int length = -1,
#9 425.7 npm ERR!        ^~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3277:7: note:   no known conversion for argument 1 from 'char*' to 'v8::Isolate*'
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In member function 'v8::Local<v8::Value> Nan::Callback::Call_(v8::Isolate*, v8::Local<v8::Object>, int, v8::Local<v8::Value>*) const':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:1479:5: warning: 'v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)' is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
#9 425.7 npm ERR!      ));
#9 425.7 npm ERR!      ^
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:195:50: note: declared here
#9 425.7 npm ERR!                  NODE_EXTERN v8::Local<v8::Value> MakeCallback(
#9 425.7 npm ERR!                                                   ^~~~~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:112:42: note: in definition of macro 'NODE_DEPRECATED'
#9 425.7 npm ERR!      __attribute__((deprecated(message))) declarator
#9 425.7 npm ERR!                                           ^~~~~~~~~~
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h:1479:5: warning: 'v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)' is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
#9 425.7 npm ERR!      ));
#9 425.7 npm ERR!      ^
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:195:50: note: declared here
#9 425.7 npm ERR!                  NODE_EXTERN v8::Local<v8::Value> MakeCallback(
#9 425.7 npm ERR!                                                   ^~~~~~~~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/node.h:112:42: note: in definition of macro 'NODE_DEPRECATED'
#9 425.7 npm ERR!      __attribute__((deprecated(message))) declarator
#9 425.7 npm ERR!                                           ^~~~~~~~~~
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In member function 'void Nan::AsyncWorker::SaveToPersistent(const char*, const v8::Local<v8::Value>&)':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:1533:64: error: no matching function for call to 'v8::Object::Set(v8::Local<v8::String>, const v8::Local<v8::Value>&)'
#9 425.7 npm ERR!      New(persistentHandle)->Set(New(key).ToLocalChecked(), value);
#9 425.7 npm ERR!                                                                 ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3961:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, v8::Local<v8::Value>, v8::Local<v8::Value>)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
#9 425.7 npm ERR!                                      ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3961:37: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3964:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, uint32_t, v8::Local<v8::Value>)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
#9 425.7 npm ERR!                                      ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3964:37: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In member function 'void Nan::AsyncWorker::SaveToPersistent(const v8::Local<v8::String>&, const v8::Local<v8::Value>&)':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:1539:42: error: no matching function for call to 'v8::Object::Set(const v8::Local<v8::String>&, const v8::Local<v8::Value>&)'
#9 425.7 npm ERR!      New(persistentHandle)->Set(key, value);
#9 425.7 npm ERR!                                           ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3961:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, v8::Local<v8::Value>, v8::Local<v8::Value>)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
#9 425.7 npm ERR!                                      ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3961:37: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3964:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, uint32_t, v8::Local<v8::Value>)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
#9 425.7 npm ERR!                                      ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3964:37: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In member function 'void Nan::AsyncWorker::SaveToPersistent(uint32_t, const v8::Local<v8::Value>&)':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:1545:44: error: no matching function for call to 'v8::Object::Set(uint32_t&, const v8::Local<v8::Value>&)'
#9 425.7 npm ERR!      New(persistentHandle)->Set(index, value);
#9 425.7 npm ERR!                                             ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3961:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, v8::Local<v8::Value>, v8::Local<v8::Value>)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
#9 425.7 npm ERR!                                      ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3961:37: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3964:37: note: candidate: 'v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, uint32_t, v8::Local<v8::Value>)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
#9 425.7 npm ERR!                                      ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3964:37: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In member function 'v8::Local<v8::Value> Nan::AsyncWorker::GetFromPersistent(const char*) const':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:1551:61: error: no matching function for call to 'v8::Object::Get(v8::Local<v8::String>)'
#9 425.7 npm ERR!          New(persistentHandle)->Get(New(key).ToLocalChecked()));
#9 425.7 npm ERR!                                                              ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4007:43: note: candidate: 'v8::MaybeLocal<v8::Value> v8::Object::Get(v8::Local<v8::Context>, v8::Local<v8::Value>)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
#9 425.7 npm ERR!                                            ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4007:43: note:   candidate expects 2 arguments, 1 provided
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4010:43: note: candidate: 'v8::MaybeLocal<v8::Value> v8::Object::Get(v8::Local<v8::Context>, uint32_t)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
#9 425.7 npm ERR!                                            ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4010:43: note:   candidate expects 2 arguments, 1 provided
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In member function 'v8::Local<v8::Value> Nan::AsyncWorker::GetFromPersistent(const v8::Local<v8::String>&) const':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:1557:55: error: no matching function for call to 'v8::Object::Get(const v8::Local<v8::String>&)'
#9 425.7 npm ERR!      return scope.Escape(New(persistentHandle)->Get(key));
#9 425.7 npm ERR!                                                        ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4007:43: note: candidate: 'v8::MaybeLocal<v8::Value> v8::Object::Get(v8::Local<v8::Context>, v8::Local<v8::Value>)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
#9 425.7 npm ERR!                                            ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4007:43: note:   candidate expects 2 arguments, 1 provided
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4010:43: note: candidate: 'v8::MaybeLocal<v8::Value> v8::Object::Get(v8::Local<v8::Context>, uint32_t)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
#9 425.7 npm ERR!                                            ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4010:43: note:   candidate expects 2 arguments, 1 provided
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In member function 'v8::Local<v8::Value> Nan::AsyncWorker::GetFromPersistent(uint32_t) const':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:1562:57: error: no matching function for call to 'v8::Object::Get(uint32_t&)'
#9 425.7 npm ERR!      return scope.Escape(New(persistentHandle)->Get(index));
#9 425.7 npm ERR!                                                          ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4007:43: note: candidate: 'v8::MaybeLocal<v8::Value> v8::Object::Get(v8::Local<v8::Context>, v8::Local<v8::Value>)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
#9 425.7 npm ERR!                                            ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4007:43: note:   candidate expects 2 arguments, 1 provided
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4010:43: note: candidate: 'v8::MaybeLocal<v8::Value> v8::Object::Get(v8::Local<v8::Context>, uint32_t)'
#9 425.7 npm ERR!    V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
#9 425.7 npm ERR!                                            ^~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:4010:43: note:   candidate expects 2 arguments, 1 provided
#9 425.7 npm ERR! In file included from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan.h: In function 'void Nan::AsyncQueueWorker(Nan::AsyncWorker*)':
#9 425.7 npm ERR! ../node_modules/nan/nan.h:1732:62: warning: cast between incompatible function types from 'void (*)(uv_work_t*)' {aka 'void (*)(uv_work_s*)'} to 'uv_after_work_cb' {aka 'void (*)(uv_work_s*, int)'} [-Wcast-function-type]
#9 425.7 npm ERR!      , reinterpret_cast<uv_after_work_cb>(AsyncExecuteComplete)
#9 425.7 npm ERR!                                                               ^
#9 425.7 npm ERR! In file included from ../node_modules/nan/nan.h:2222,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../node_modules/nan/nan_object_wrap.h: In member function 'void Nan::ObjectWrap::MakeWeak()':
#9 425.7 npm ERR! ../node_modules/nan/nan_object_wrap.h:67:18: error: 'class Nan::Persistent<v8::Object>' has no member named 'MarkIndependent'
#9 425.7 npm ERR!      persistent().MarkIndependent();
#9 425.7 npm ERR!                   ^~~~~~~~~~~~~~~
#9 425.7 npm ERR! In file included from ../src/database.cc:3:
#9 425.7 npm ERR! ../src/database.cc: In member function 'void node_sqlite3::Database::Process()':
#9 425.7 npm ERR! ../src/macros.h:91:13: error: no matching function for call to 'v8::String::Concat(v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!              ),                                                                 \
#9 425.7 npm ERR!              ^
#9 425.7 npm ERR! ../src/database.cc:40:9: note: in expansion of macro 'EXCEPTION'
#9 425.7 npm ERR!          EXCEPTION(Nan::New("Database handle is closed").ToLocalChecked(), SQLITE_MISUSE, exception);
#9 425.7 npm ERR!          ^~~~~~~~~
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note: candidate: 'static v8::Local<v8::String> v8::String::Concat(v8::Isolate*, v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!    static Local<String> Concat(Isolate* isolate, Local<String> left,
#9 425.7 npm ERR!                         ^~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! In file included from ../src/database.cc:3:
#9 425.7 npm ERR! ../src/database.cc: In member function 'void node_sqlite3::Database::Schedule(node_sqlite3::Database::Work_Callback, node_sqlite3::Database::Baton*, bool)':
#9 425.7 npm ERR! ../src/macros.h:91:13: error: no matching function for call to 'v8::String::Concat(v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!              ),                                                                 \
#9 425.7 npm ERR!              ^
#9 425.7 npm ERR! ../src/database.cc:88:9: note: in expansion of macro 'EXCEPTION'
#9 425.7 npm ERR!          EXCEPTION(Nan::New("Database is closed").ToLocalChecked(), SQLITE_MISUSE, exception);
#9 425.7 npm ERR!          ^~~~~~~~~
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note: candidate: 'static v8::Local<v8::String> v8::String::Concat(v8::Isolate*, v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!    static Local<String> Concat(Isolate* isolate, Local<String> left,
#9 425.7 npm ERR!                         ^~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! ../src/database.cc: In static member function 'static void node_sqlite3::Database::Work_BeginOpen(node_sqlite3::Database::Baton*)':
#9 425.7 npm ERR! ../src/database.cc:145:55: warning: cast between incompatible function types from 'void (*)(uv_work_t*)' {aka 'void (*)(uv_work_s*)'} to 'uv_after_work_cb' {aka 'void (*)(uv_work_s*, int)'} [-Wcast-function-type]
#9 425.7 npm ERR!          &baton->request, Work_Open, (uv_after_work_cb)Work_AfterOpen);
#9 425.7 npm ERR!                                                        ^~~~~~~~~~~~~~
#9 425.7 npm ERR! ../src/database.cc:144:9: warning: unused variable 'status' [-Wunused-variable]
#9 425.7 npm ERR!      int status = uv_queue_work(uv_default_loop(),
#9 425.7 npm ERR!          ^~~~~~
#9 425.7 npm ERR! In file included from ../src/database.cc:3:
#9 425.7 npm ERR! ../src/database.cc: In static member function 'static void node_sqlite3::Database::Work_AfterOpen(uv_work_t*)':
#9 425.7 npm ERR! ../src/macros.h:91:13: error: no matching function for call to 'v8::String::Concat(v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!              ),                                                                 \
#9 425.7 npm ERR!              ^
#9 425.7 npm ERR! ../src/database.cc:179:9: note: in expansion of macro 'EXCEPTION'
#9 425.7 npm ERR!          EXCEPTION(Nan::New(baton->message.c_str()).ToLocalChecked(), baton->status, exception);
#9 425.7 npm ERR!          ^~~~~~~~~
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note: candidate: 'static v8::Local<v8::String> v8::String::Concat(v8::Isolate*, v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!    static Local<String> Concat(Isolate* isolate, Local<String> left,
#9 425.7 npm ERR!                         ^~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! ../src/database.cc: In static member function 'static void node_sqlite3::Database::Work_BeginClose(node_sqlite3::Database::Baton*)':
#9 425.7 npm ERR! ../src/database.cc:231:56: warning: cast between incompatible function types from 'void (*)(uv_work_t*)' {aka 'void (*)(uv_work_s*)'} to 'uv_after_work_cb' {aka 'void (*)(uv_work_s*, int)'} [-Wcast-function-type]
#9 425.7 npm ERR!          &baton->request, Work_Close, (uv_after_work_cb)Work_AfterClose);
#9 425.7 npm ERR!                                                         ^~~~~~~~~~~~~~~
#9 425.7 npm ERR! ../src/database.cc:230:9: warning: unused variable 'status' [-Wunused-variable]
#9 425.7 npm ERR!      int status = uv_queue_work(uv_default_loop(),
#9 425.7 npm ERR!          ^~~~~~
#9 425.7 npm ERR! In file included from ../src/database.cc:3:
#9 425.7 npm ERR! ../src/database.cc: In static member function 'static void node_sqlite3::Database::Work_AfterClose(uv_work_t*)':
#9 425.7 npm ERR! ../src/macros.h:91:13: error: no matching function for call to 'v8::String::Concat(v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!              ),                                                                 \
#9 425.7 npm ERR!              ^
#9 425.7 npm ERR! ../src/database.cc:259:9: note: in expansion of macro 'EXCEPTION'
#9 425.7 npm ERR!          EXCEPTION(Nan::New(baton->message.c_str()).ToLocalChecked(), baton->status, exception);
#9 425.7 npm ERR!          ^~~~~~~~~
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note: candidate: 'static v8::Local<v8::String> v8::String::Concat(v8::Isolate*, v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!    static Local<String> Concat(Isolate* isolate, Local<String> left,
#9 425.7 npm ERR!                         ^~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! ../src/database.cc: In static member function 'static Nan::NAN_METHOD_RETURN_TYPE node_sqlite3::Database::Configure(Nan::NAN_METHOD_ARGS_TYPE)':
#9 425.7 npm ERR! ../src/database.cc:351:9: error: no matching function for call to 'v8::String::Concat(v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!          )));
#9 425.7 npm ERR!          ^
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note: candidate: 'static v8::Local<v8::String> v8::String::Concat(v8::Isolate*, v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!    static Local<String> Concat(Isolate* isolate, Local<String> left,
#9 425.7 npm ERR!                         ^~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! ../src/database.cc:351:11: error: return-statement with a value, in function returning 'Nan::NAN_METHOD_RETURN_TYPE' {aka 'void'} [-fpermissive]
#9 425.7 npm ERR!          )));
#9 425.7 npm ERR!            ^
#9 425.7 npm ERR! ../src/database.cc: In static member function 'static void node_sqlite3::Database::Work_BeginExec(node_sqlite3::Database::Baton*)':
#9 425.7 npm ERR! ../src/database.cc:526:55: warning: cast between incompatible function types from 'void (*)(uv_work_t*)' {aka 'void (*)(uv_work_s*)'} to 'uv_after_work_cb' {aka 'void (*)(uv_work_s*, int)'} [-Wcast-function-type]
#9 425.7 npm ERR!          &baton->request, Work_Exec, (uv_after_work_cb)Work_AfterExec);
#9 425.7 npm ERR!                                                        ^~~~~~~~~~~~~~
#9 425.7 npm ERR! ../src/database.cc:525:9: warning: unused variable 'status' [-Wunused-variable]
#9 425.7 npm ERR!      int status = uv_queue_work(uv_default_loop(),
#9 425.7 npm ERR!          ^~~~~~
#9 425.7 npm ERR! In file included from ../src/database.cc:3:
#9 425.7 npm ERR! ../src/database.cc: In static member function 'static void node_sqlite3::Database::Work_AfterExec(uv_work_t*)':
#9 425.7 npm ERR! ../src/macros.h:91:13: error: no matching function for call to 'v8::String::Concat(v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!              ),                                                                 \
#9 425.7 npm ERR!              ^
#9 425.7 npm ERR! ../src/database.cc:557:9: note: in expansion of macro 'EXCEPTION'
#9 425.7 npm ERR!          EXCEPTION(Nan::New(baton->message.c_str()).ToLocalChecked(), baton->status, exception);
#9 425.7 npm ERR!          ^~~~~~~~~
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note: candidate: 'static v8::Local<v8::String> v8::String::Concat(v8::Isolate*, v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!    static Local<String> Concat(Isolate* isolate, Local<String> left,
#9 425.7 npm ERR!                         ^~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! ../src/database.cc: In static member function 'static void node_sqlite3::Database::Work_BeginLoadExtension(node_sqlite3::Database::Baton*)':
#9 425.7 npm ERR! ../src/database.cc:626:104: warning: cast between incompatible function types from 'void (*)(uv_work_t*)' {aka 'void (*)(uv_work_s*)'} to 'uv_after_work_cb' {aka 'void (*)(uv_work_s*, int)'} [-Wcast-function-type]
#9 425.7 npm ERR!          &baton->request, Work_LoadExtension, reinterpret_cast<uv_after_work_cb>(Work_AfterLoadExtension));
#9 425.7 npm ERR!                                                                                                         ^
#9 425.7 npm ERR! ../src/database.cc:625:9: warning: unused variable 'status' [-Wunused-variable]
#9 425.7 npm ERR!      int status = uv_queue_work(uv_default_loop(),
#9 425.7 npm ERR!          ^~~~~~
#9 425.7 npm ERR! In file included from ../src/database.cc:3:
#9 425.7 npm ERR! ../src/database.cc: In static member function 'static void node_sqlite3::Database::Work_AfterLoadExtension(uv_work_t*)':
#9 425.7 npm ERR! ../src/macros.h:91:13: error: no matching function for call to 'v8::String::Concat(v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!              ),                                                                 \
#9 425.7 npm ERR!              ^
#9 425.7 npm ERR! ../src/database.cc:659:9: note: in expansion of macro 'EXCEPTION'
#9 425.7 npm ERR!          EXCEPTION(Nan::New(baton->message.c_str()).ToLocalChecked(), baton->status, exception);
#9 425.7 npm ERR!          ^~~~~~~~~
#9 425.7 npm ERR! In file included from /root/.cache/node-gyp/16.20.2/include/node/node.h:73,
#9 425.7 npm ERR!                  from ../node_modules/nan/nan.h:49,
#9 425.7 npm ERR!                  from ../src/database.h:10,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note: candidate: 'static v8::Local<v8::String> v8::String::Concat(v8::Isolate*, v8::Local<v8::String>, v8::Local<v8::String>)'
#9 425.7 npm ERR!    static Local<String> Concat(Isolate* isolate, Local<String> left,
#9 425.7 npm ERR!                         ^~~~~~
#9 425.7 npm ERR! /root/.cache/node-gyp/16.20.2/include/node/v8.h:3515:24: note:   candidate expects 3 arguments, 2 provided
#9 425.7 npm ERR! In file included from ../src/database.h:12,
#9 425.7 npm ERR!                  from ../src/database.cc:4:
#9 425.7 npm ERR! ../src/async.h: In instantiation of 'Async<Item, Parent>::Async(Parent*, Async<Item, Parent>::Callback) [with Item = std::__cxx11::basic_string<char>; Parent = node_sqlite3::Database; Async<Item, Parent>::Callback = void (*)(node_sqlite3::Database*, std::__cxx11::basic_string<char>*)]':
#9 425.7 npm ERR! ../src/database.cc:391:59:   required from here
#9 425.7 npm ERR! ../src/async.h:29:52: warning: cast between incompatible function types from 'void (*)(uv_async_t*, int)' {aka 'void (*)(uv_async_s*, int)'} to 'uv_async_cb' {aka 'void (*)(uv_async_s*)'} [-Wcast-function-type]
#9 425.7 npm ERR!          uv_async_init(uv_default_loop(), &watcher, reinterpret_cast<uv_async_cb>(listener));
#9 425.7 npm ERR!                                                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#9 425.7 npm ERR! ../src/async.h: In instantiation of 'Async<Item, Parent>::Async(Parent*, Async<Item, Parent>::Callback) [with Item = node_sqlite3::Database::ProfileInfo; Parent = node_sqlite3::Database; Async<Item, Parent>::Callback = void (*)(node_sqlite3::Database*, node_sqlite3::Database::ProfileInfo*)]':
#9 425.7 npm ERR! ../src/database.cc:429:65:   required from here
#9 425.7 npm ERR! ../src/async.h:29:52: warning: cast between incompatible function types from 'void (*)(uv_async_t*, int)' {aka 'void (*)(uv_async_s*, int)'} to 'uv_async_cb' {aka 'void (*)(uv_async_s*)'} [-Wcast-function-type]
#9 425.7 npm ERR! ../src/async.h: In instantiation of 'Async<Item, Parent>::Async(Parent*, Async<Item, Parent>::Callback) [with Item = node_sqlite3::Database::UpdateInfo; Parent = node_sqlite3::Database; Async<Item, Parent>::Callback = void (*)(node_sqlite3::Database*, node_sqlite3::Database::UpdateInfo*)]':
#9 425.7 npm ERR! ../src/database.cc:470:62:   required from here
#9 425.7 npm ERR! ../src/async.h:29:52: warning: cast between incompatible function types from 'void (*)(uv_async_t*, int)' {aka 'void (*)(uv_async_s*, int)'} to 'uv_async_cb' {aka 'void (*)(uv_async_s*)'} [-Wcast-function-type]
#9 425.7 npm ERR! make: *** [node_sqlite3.target.mk:133: Release/obj.target/node_sqlite3/src/database.o] Error 1
#9 425.7 npm ERR! gyp ERR! build error 
#9 425.7 npm ERR! gyp ERR! stack Error: `make` failed with exit code: 2
#9 425.7 npm ERR! gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:201:23)
#9 425.7 npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
#9 425.7 npm ERR! gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:293:12)
#9 425.7 npm ERR! gyp ERR! System Linux 3.10.0-1127.10.1.el7.x86_64
#9 425.7 npm ERR! gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--module=/usr/src/app/node_modules/sqlite3/lib/binding/node-v93-linux-x64/node_sqlite3.node" "--module_name=node_sqlite3" "--module_path=/usr/src/app/node_modules/sqlite3/lib/binding/node-v93-linux-x64"
#9 425.7 npm ERR! gyp ERR! cwd /usr/src/app/node_modules/sqlite3
#9 425.7 npm ERR! gyp ERR! node -v v16.20.2
#9 425.7 npm ERR! gyp ERR! node-gyp -v v9.1.0
#9 425.7 npm ERR! gyp ERR! not ok
#9 425.7 
#9 425.7 npm ERR! A complete log of this run can be found in:
#9 425.7 npm ERR!     /root/.npm/_logs/2024-07-31T07_30_00_195Z-debug-0.log
#9 ERROR: process "/bin/sh -c npm install && npm cache clean --force" did not complete successfully: exit code: 1
------
 > [5/6] RUN npm install && npm cache clean --force:
425.7 npm ERR! gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:293:12)
425.7 npm ERR! gyp ERR! System Linux 3.10.0-1127.10.1.el7.x86_64
425.7 npm ERR! gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--module=/usr/src/app/node_modules/sqlite3/lib/binding/node-v93-linux-x64/node_sqlite3.node" "--module_name=node_sqlite3" "--module_path=/usr/src/app/node_modules/sqlite3/lib/binding/node-v93-linux-x64"
425.7 npm ERR! gyp ERR! cwd /usr/src/app/node_modules/sqlite3
425.7 npm ERR! gyp ERR! node -v v16.20.2
425.7 npm ERR! gyp ERR! node-gyp -v v9.1.0
425.7 npm ERR! gyp ERR! not ok
425.7 
425.7 npm ERR! A complete log of this run can be found in:
425.7 npm ERR!     /root/.npm/_logs/2024-07-31T07_30_00_195Z-debug-0.log
------
v1_Dockerfile:7
--------------------
   5 |     
   6 |     COPY package.json /usr/src/app
   7 | >>> RUN npm install && npm cache clean --force
   8 |     
   9 |     COPY . /usr/src/app
--------------------
ERROR: failed to solve: process "/bin/sh -c npm install && npm cache clean --force" did not complete successfully: exit code: 1
Duration: 8:31.07